Deep Missions: Skadenga Saga (#6263)
endless-sky/endless-sky: data/human/deep missions.txt

			`	Hjlod shrugs. "Maybe they fall in ice. Maybe they get eaten by Ice-Slug. Maybe avalanche fall on them. Maybe fall on own veapon and die. Nobody ever find them. Tour-guide book say that some night you can see their ghost haunting ice. Tour-guide book dumb."`
			`	"Any more question?"`
				goto questions
			label ice
			`	"Very safe. Except when not. Ice always frozen, but sometime very thin. I can tell. You cannot. Trust Hjlod and you not drown and get eaten by Ice-Slug."`
			` 	"You have more question?"`
				goto questions
			label monster
			`	"Every world have animal. Some animal big, some small. Voolly octopus, mostly small. Size of this Ice-Crawler, only. Vorst in Glaesisvellir ice-plains is Ice-Slug. They can squeeze through tiniest hole in ice. This take long time, for they are as big as house. They have no teeth, too hard to squeeze through little hole, but they have acid spit. Put hole in Ice-Crawler, easy. They eat many foolish researcher and tourist."`
			`	She smiles at you. "You're not foolish tourist. You are with Hjlod. You are safe."`
			`	"Any more question?"`
				goto questions
			label noquestion

> 			`	"Is smart, not to ask question. Hjlod tell you best parts only." Hjlod gives you a quick grin and swivels back into her makeshift cockpit. "Enough ice-plain, now ve go to next stop."`

Reviewer: This response will still appear even if the player has asked questions, which makes the `Is smart, not to ask question` bit feel a little weird. Consider rephrasing this line to avoid that issue.

Author: I was thinking that, in the event you asked questions, Hjlod here is commenting on the fact it is smart not to ask anymore questions.

			`	"I think-" Hjlod starts to say when a mountain of snow crashes into the Ice-Crawler with deafening force. All sense of travelling distance is lost as the snow blots out the windows and the crawler is dragged bumping and twisting over uneven terrain. You and Hjlod hold tight as you're tossed around until, eventually, the crawler grinds up against something reasonably solid and comes suddenly to a rest, partially on its side.`
			`	A few moments later, the rumbling finishes passing you by.`
			choice
				`	(Unbuckle your seatbelt and get yourself up.)`
			`	By the time you are on your feet, Hjlod has already gathered up numerous supplies and is preparing to open the hatch and dig her way to the surface. She looks at you. "Nothing to vorry about. Happens all time. Vell, not really. First time this happen, but Hjlod not let you die, probably. I can lead us to settlement not far from here."`
			`	She tosses you an old, scruffy jacket. "Valking in cold, good for health. Maybe not as far as ve must go, but you laugh about later. Good story to tell back home."`
			choice
				`	(Put on the jacket and follow Hjlod out onto the ice.)`
					goto jacket

> 				`	"No way. I am done with this. I am calling my ship, or fixing this Ice-Crawler. Either way, I am not going anywhere."`

Reviewer: Considering this choice leads to either death or entirely missing the mission chain, this feels maybe a little too easy to choose. Can we add some sort of confirmation choice and/or more that emphasizes trying to fix this yourself is a bad idea?

Author: Whenever the player chooses to return to the spaceport/or their ship, and not continue, they quest ends. That's in the players hand. If they want to continue, they can, and if they want to get out of this, they can. I don't think the player needs their hand held here. 

Technically, trying to fix the crawler yourself can result in the best outcome as well (assuming you don't blow yourself up).

Reviewer: Fair enough!

				decline
			label fixme
			`	After hours of tinkering with the guts of the machine, you've managed to repair most of it, and all it cost you was one bad burn on your arm. Now, what it comes down to is a sequence of wires. If you can plug these in correctly, the Ice-Crawler should start back up.`
			choice
				`	(White, green, blue.)`
					goto boom
				`	(Green, white, blue.)`
					goto greatsuccess
				`	(Blue, green, white.)`
					goto boom
			label boom
			` 	You're confident that this is the correct sequence of wires. You plug each one in and start the Ice-Crawler up. The vehicle hums to life.`
			choice
				`	(Pump your fist.)`
				`	(Cheer.)`
					goto cheer
			` 	In a moment of levity, you pump your fist into the air.`
			`	Unfortunately, your fist does not make it very far. The Ice-Crawler, with its criss-crossed wires, has a critical energy build up that results in an instantaneous, completely deadly explosion.`

> 			scene "scene/smeer"

Reviewer: This picture feels unnecessary to me, personally.

Author: I felt blowing yourself up was dramatic enough that some sort of 'end-game' picture would be fitting. I can however remove it, and all the other scenes, if they are felt in appropriate.

Reviewer: Any other folks have opinions on this?

Reviewer 2: I don't mind the fact that there are images, but the fact that these images are paintings feels out of place in ES. Every landscape image in this game is a real photograph, and I think that should hold here too.

Author: It has been replaced with a photograph. If you still think it is unnecessary I will remove it entirely.

Reviewer: Think I'm good with that!

			`	The Ice-Crawler, your entire body, your fist, and more than a couple hundred tons of snow are all incinerated in a fraction of a second.`
				die
			label cheer
			`	The sense of accomplishment inside you is so overwhelming that you find yourself unable to contain it. A cry of pure joy leaps out of your mouth. Or begins too. Unfortunately, it doesn't get too far. As the Ice-Crawler, with its criss-crossed wires, has a critical energy build up that results in an instantaneous, completely deadly explosion.`

> 			scene "scene/smeer"

Reviewer: Same as above.

Author: I have replaced this image with a new one. Does it still feel unnecessary? If you do not like the new picture I will remove the /smeer image entirely.

				set "Asgard"
			`	You answer, "On Asgard, there is a great market built around their God-Stone, and many new homes in need of people to fill them. The people there have lost their way and they need the Skadenga to remind them of who they were."`
			`	Hjlod nods. "The Skadenga have alvays been guides, pathfinders. I vill tell Ondurdis.`
			label end
			`	"Vhen you were gone, thing, or assembly, vas held. Ondurdis called in all Skadenga from all villages across Nifel. You must find transportation for 2,200 folk. That is what Onderduis told me to tell you."`
			`	You frown and ask, "Did she provide money?"`
			`	Hjlod shakes her head. "No. Ve could not afford it. You must go to Valhalla and convince head of Transport Union to support us." She squeezes your shoulder. "Thank you for help. For finding us new home. Ve need it. I must report now to Ondurdis. Skade be vith you." She turns and leaves.`
				accept
	on complete
		conversation
			`	Outside your ship, you are met by a representative of the Transport Union. "We have been expecting you, Captain <last>." They escort you to an elevator and up to the penthouse of the tallest building in Valhalla's capital of Himinbjorg."`

> 			scene "scene/tower"

Reviewer: This picture doesn't feel too tonally consistent with ES to me. I'd be curious what others besides myself think.

> 			`	Three youngish men adorned in trending clothes smile as you enter. "Ah, Captain <last>, what a pleasure to finally meet you. I am Kaiden, and these are my associates Braiden and Grayden. We are executives of the Transport Union. Obviously, we were made aware you would be coming and are looking forward toward creating a lasting and beneficial relationship with you and your... clients.`

Reviewer: These characters feels too... "on the nose" bad/evil to me in a way that doesn't blend well with the rest of ES. The way I see most of the game, the vast majority of characters (maybe excluding alphas) are at conflict not because there's good guys and bad guys, but just people who have different points of view. I don't get that same feeling here. I doubt anyone would think to themselves, "I agree with these people," or even, "I get where they're coming from." The thing that especially stands out is the derogatory tone/words used toward the Skadenga (`rustic primitives`, `savages`, etc). I'd suggest some rewriting that maybe tones down the whole "17th century colonists vs. natives" vibe.

Author: As far fetched as it seems, I would argue that decisions/thinking like those can be found in powerful corporate executive circles. There are poor people, native people, and just regular people today who suffer because of short-sighted / profit motived decisions. It's not a 17th century problem, it's a today problem, and will be a tomorrow problem. 

I can remove some of derogatory references if you really think its over top, but I think you're giving humanity way too much credit if think it's unbelievable.

Reviewer: I gotcha. I still do get the vibe that it's over the top for me - I think a good compromise would be removing some of the derogatory references, as you suggested. The ones I'd specifically suggest rewording/taking out are uses of `sad savages` and `misfit primitives`. There are a couple instances too outside of this specific conversation with Carla, the narration, and others using them - consider rewording those too.

Author: Alright. I removed some instances of 'primitive' and 'savage' from the text. This includes the instances you brought up, plus a few others.

Reviewer: Some of the changes I like, some are still "eh" to me, personally. I was more hoping you'd reword into something less aggressive rather than at the same level of hostility. I'll throw in a few suggestions myself and see if you like them.

Author: I have accepted your suggestions.

event "mani refit"

mission "Mani Refit"
	name "Mani Refit"
	to offer
		has "event: mani refit"
		has "Home for Skadenga 5: done"
	description "Head back to <destination> and meet up with the Transport Union's Fleet."
	source
		government "Republic"
	destination "Mani"
	on offer
		dialog
			`Your inbox blinks with a new message from the Transport Union.`
			`	'The fleet is ready for its inspirational journey. Please return to Mani.`
			`		-Jayden, Mani Administrator, Transport Union`

mission "Home for Skadenga 6"
	landing
	name "Transport the Skadenga"
	description `Escort a fleet of freighters, refitted for carrying primitives, to <destination>.`
	source "Mani"
	destination "Nifel"
	to offer
		has	"Asgard2"
	on offer
		conversation
			`You arrive at the sapceport to see a Union Executive overseeing the final preparations of his fleet. After a moment, the last crewmembers leave his side and board their ships. He walks over to you. "The fleet is ready to launch. Please, try to bring them back to me in one piece."`
			choice

> 				`	"Let's get this over with."`

Reviewer: Some extra dialogue options here would be appreciated. Doesn't have to lead to different responses or anything.

Author: Added a few more options.

	on offer
		payment 300000
		event "asgard hjlod" 2000
		conversation
			`You receive a message from your bank that <payment> have been deposited into your account from the Census and Culture Bureau of The Deep. The following memo is attached:`
			`	"For heroic and selfless service to the ideals and prosperity of The Deep."`
				decline

event "asgard hjlod"

mission "Skadenga Call 4"
	name "Hjlod's goodbye"
	invisible
	destination "Nifel"
	to offer
		has "Skadenga Call 2: done"
		has "event: nifelhjlod"
	on complete
		event "hjlod waits again" 360
		conversation
			`A familiar woman is waiting for you outside your ship. Although it's been more than seven years since you've last seen her, you recognize Hjlod instantly. Her eyes are brighter than when last you parted, and she has the same lopsided grin that you remember. "<first>, I saw your ship landing, and thought I vould say hello."`
			`	You ask her how she has been, and she tells you that the small community of Skadenga have rebuilt and thrived here. That there has even been a small amount of immigration to their tiny town. That word had gotten out about their desperate escape from Asgard.`

> 			`	She tells you that she forgives your past actions, and hopes you find whatever it is you are looking for among the stars. She even hugs you before she leaves. Although she might forgive you, you wonder if you can ever forgive yourself...`

Reviewer: `you wonder if you can ever forgive yourself` pushes a perspective the player themselves might not have a little too much for me. Consider finding a different way to finish this line.

Author: Removed the line.

	on offer
		event "hroar message" 13
		conversation
			`The floor of the spaceport is covered in the dirty red dust of hundreds of shuffling feet. Prospectors, miners, drillers, and all their attendant associates are intermixed here, gossiping about the latest finds, accidents, and hauls. Every person, either saturated in red earth or immaculately clean, creates a visual aid to boundaries of class, paycheck, and leadership.`
			scene "scene/redrocks"
			`	The Red Rock Saloon is located deep in the trade-craft section of the spaceport, far enough from the tourist areas to discourage anyone but the locals and those with business to enter.`
			choice
				`	(Enter the Red Rock Saloon.)`
			`	The doors swing open, revealing a dimly lit room filled with rough unsavory people. Piano music is playing on a cheap speaker, and the eyes of the patrons slide to get a good look at you without turning their heads. There is an expectant pause. You get the feeling they are waiting for you to either announce something or leave.`
			choice
				`	"I'm looking for Hroar."`
			`	For a moment nothing happens, and then a gruff voice spirals out from a dark corner. "Back here."`

> 			scene "scene/hroar"

Reviewer: I'm probably the most iffy on this picture. Nothing else I can think of in ES is hand-drawn. Again, I'd be curious to see what other think. Maybe it's just me.

Author: I will be replacing most, if not all, of the pictures. @roadrunner56 has offered to help in this.

Author: All images have been replaced with photographs.

			`	She then nods at you. "If you could transport family members of deceased to return their stones, it would mean much to everyone here. We would pay you <payment>."`
			choice
				`	"Of course."`
					goto yes
				`	"I'll do it for free."`
					goto yesyes
				`	"No thanks."`
			`	You explain to them you have more pressing matters to attend to and cannot be a taxi service for their grieving. They graciously accept your excuses, and you leave back to your ship.`
				decline
			label yesyes
			`	The Ondurdis shakes her head. "You have already done too much for us for too little pay. We can afford to pay you, and we will."`
			label yes
			`	"The five Stone-Bearers will accompany you back to your ship. May Skade and Ran watch over your journey." The Ondurdis turns this into a toast, which everyone gladly participates in.`
			`	Hjlod walks with you and your passengers back to your ship. "Be well, <first>."`
				accept
	on complete
		payment 50000
		event "hjlod waits" 360

> mission "Stones of Skadenga 1"

Reviewer: Is this mission really necessary? The norm for the game is that "out-and-back" missions pay upfront, and that's what I assumed was the case when I read the preceding mission. I just thought bringing them back to Windblain was part of the deal for the amount they were offering to pay me.
I'd personally just combine this with the one above into a stopover mission and remove the option to refuse to take them back, unless you're dead-set on keeping it. If you are, I'd be curious what the reasoning is.

Author: I have combined one of these missions, but the very first one is still split into two parts, however I removed the option to decline it and leave the stone-bearers stranded. The reason I've left it in two parts is because there is a fair bit going on and I am not confident enough in my abilities to encapsulate it all in a single stopover mission. However, with the decline option removed, it functions very similar to a stopover mission now.

			`	The mourners ask if you can take them back to <planet>. They have already paid you all their money, and cannot afford any  more for a ride home.`
			choice
				`	(Return the mourning Skadenga to <planet>.)`
					goto home
				`	(Leave them here.)`
			`	You let them know that you only agreed to take them here, not bring them back. You shut your hatch and leave them to fend for themselves.`
				decline
			label home
			`	They thank you and board your ship.`
				accept
	on complete
		dialog
			`The Skadenga are happy to be home. The churning seas of <planet> lift their spirits, and they head out into the sea-sweet air.`

mission "Stones of Skadenga 4"
	invisible
	source "Windblain"
	to offer
		has "Stones of Skadenga 3: done"
	on offer
		conversation
			`You're surprised to see the Ondurdis walking toward you.`
			`	She nods politely before speaking. "<first>, I just wanted to thank you for ferrying our Stone-Bearers to <planet> and back. I have added this task to the job board so you can more easily assist in the future. I've also let them know they need enough money to pay for a return trip." The Ondurdis touches your shoulder, and then walks away.`
				decline

> mission "Stone-Bearers to Nifel"

Reviewer: This could also just be combined with the mission below into one stopover mission. I don't think it makes sense for a variable amount of passengers to go just one way anyways, unless other captains are taking them too.

Author: I have combined it with the other mission, as you requested.

		`	There's an old-style book there, bound up in a ice-blue cover, titled: "The Skakdenga Saga: Against All Odds, by Hjlod Hrimnirsdottir."`
		choice
			`	(Ignore the book and move on.)`
				goto ignore
			`	(Pick the book up.)`
			branch "windblain"
				has "Stones of Skadenga 4: done"
			`	The book feels solid in your hands, and you scan through it. There are chapters about near death survival on Nifel, the subtle hostility of Asgard, the sudden death of the old Ondurdis, who you learn was named Eymani, and the triumph of Hjlod and the survivors on Nifel.`
				goto end
			label windblain
			`	The book feels solid in your hands, and you scan through it. There are chapters about near death survival on Nifel, corrupt Deep officials, the colorful haulers on Muspel, and the ultimate triumph on Windblain.`
			label end
			`	You place the book back on its stand as the attendant notices you. "It's pretty popular, that one. I hear it started this whole 'back-to-nature' primalism cult." He shrugs. "Whatever sells, ya know. Anyways, you buying?"`
			`	You decline. You already know that story.`
				decline
			label ignore
			`	You keep walking. You already know that story.`
				decline

> mission "The Angry Letter"

Reviewer: This feels like setup for a future mission chain. If that's the case, I'd suggest removing it from this PR and including it with that mission chain whenever it's PRed.

Author: Removed it.

			`	She smiles at you, "You're not foolish tourist. You are with Hjlod. You are safe."`
			`	"Now, no more question." With that Hjlod swivels back into her cockpit and begins plotting a new course for the Ice-Crawler.`
			scene "scene/iceplains2"
			choice
				`	(You look out at the bright Ice-Plains as they slowly disappear into the dull horizon.)`
					goto rock
			label monster2
			`	"Every world have animal. Some animal big, some small. Voolly Octopus, mostly small. Size of this Ice-Crawler, only. Vorst in Glaesisvellir ice-plains is Ice-Slug. They can squeeze through tiniest hole in ice. This take long time, for they are as big as house. They have no teeth, too hard to squeeze through little hole, but they have acid spit. Put hole in Ice-Crawler, easy. They eat many foolish researcher and tourist."`
			`	She smiles at you. "You're not foolish tourist. You are with Hjlod. You are safe."`
			`	"You have anymore question?"`
			choice
				`	"Nope. No questions."`
					goto noquestion
				`	"What happened to the researchers?"`
					goto researcher21

> 			label researcher21
> 			`	"Long time ago. Researcher convoy look for bacterial pharmaceutical. Many researcher convoy in area. Most smart use native pathfinder, like me. This convoy, they not so smart, they say pathfinder waste of money. They go out in ice alone and they never come back."`
> 			`	Hjlod shrugs, "Maybe they fall in ice. Maybe they get eaten by Ice-Slug. Maybe avalanche fall on them. Maybe fall on own veapon and die. Nobody ever find them. Tour-guide book say that some night you can see their ghost haunting ice. Tour-guide book is dumb."`
> 			`	"Now, no more question." With that Hjlod swivels back into her cockpit and begins plotting a new course for the Ice-Crawler.`
> 			scene "scene/iceplains2"
> 			choice
> 				`	(You look out at the bright Ice-Plains as they slowly disappear into the dull horizon.)`
> 					goto rock
> 			label monster
> 			`	"Every world have animal. Some animal big, some small. Voolly Octopus, mostly small. Size of this Ice-Crawler, only. Vorst in Glaesisvellir ice-plains is Ice-Slug. They can squeeze through tiniest hole in ice. This take long time, for they are as big as house. They have no teeth, too hard to squeeze through little hole, but they have acid spit. Put hole in Ice-Crawler, easy. They eat many foolish researcher and tourist."`
> 			`	She smiles at you. "You're not foolish tourist. You are with Hjlod. You are safe."`
> 			`	"Anymore question?"`
> 			choice
> 				`	"Nope. No questions."`
> 					goto noquestion
> 				`	"What happened to the researchers?"`
> 					goto researcher3
> 				`	"Is it safe to drive over the ice?"`
> 					goto ice3
> 			label researcher3
> 			`	"Long time ago. Researcher convoy look for bacterial pharmaceutical. Many researcher convoy in area. Most smart use native pathfinder, like me. This convoy, they not so smart, they say pathfinder waste of money. They go out in ice alone and they never come back."`
> 			`	Hjlod shrugs. "Maybe they fall in ice. Maybe they get eaten by Ice-Slug. Maybe avalanche fall on them. Maybe fall on own veapon and die. Nobody ever find them. Tour-guide book say that some night you can see their ghost haunting ice. Tour-guide book is dumb."`
> 			`	"You have more question?"`
> 			choice
> 				`	"Nope. No questions."`
> 					goto noquestion
> 				`	"Is it safe to drive over the ice?"`
> 					goto ice31
> 			label ice31
> 			`	"Very safe. Except when not. Ice always frozen, but sometime very thin. I can tell. You cannot. Trust Hjlod and you not drown and get eaten by Ice-Slug."`
> 			`	"Now, no more question." With that Hjlod swivels back into her cockpit and begins plotting a new course for the Ice-Crawler.`
> 			scene "scene/iceplains2"
> 			choice
> 				`	(You look out at the bright Ice-Plains as they slowly disappear into the dull horizon.)`
> 					goto rock
> 			label ice3
> 			`	"Very safe. Except when not. Ice always frozen, but sometime very thin. I can tell. You cannot. Trust Hjlod and you not drown and get eaten by Ice-Slug."`
> 			`	"You have more question?"`
> 			choice
> 				`	"Nope. No questions."`
> 					goto noquestion
> 				`	"What happened to the researchers?"`
> 					goto researcher31
> 			label researcher31
> 			`	"Long time ago. Researcher convoy look for bacterial pharmaceutical. Many researcher convoy in area. Most smart use native pathfinder, like me. This convoy, they not so smart, they say pathfinder waste of money. They go out in ice alone and they never come back."`
> 			`	Hjlod shrugs. "Maybe they fall in ice. Maybe they get eaten by Ice-Slug. Maybe avalanche fall on them. Maybe fall on own veapon and die. Nobody ever find them. Tour-guide book say that some night you can see their ghost haunting ice. Tour-guide book is dumb."`
> 			`	"Now, no more question." With that Hjlod swivels back into her cockpit and begins plotting a new course for the Ice-Crawler.`
> 			scene "scene/iceplains2"
> 			choice
> 				`	(You look out at the bright Ice-Plains as they slowly disappear into the dull horizon.)`
> 					goto rock
> 			label noquestion
> 			`	"Is smart, not to ask question. Hjlod tell you best parts only." Hjlod gives you a quick grin and swivels back into her makeshift cockpit.`
> 			`	"Enough ice-plain, now ve go to next stop."`
> 			scene "scene/iceplains2"
> 			choice
> 				`	(You look out at the bright Ice-Plains as they slowly disappear into the dull horizon.)`
> 					goto rock
> 			label rock
> 			`	The Ice-Crawler plows through snow, hill, mountain, and tundra for another hour or two before a massive and oddly shaped rock passes into view. Hjlod stops the Ice-Crawler so you can look at it.`

Reviewer 3: ```suggestion
			`	Hjlod slows down and looks back at you. "You have question?"`
			label questions
			choice
				`	"Nope. No questions."`
					goto noquestion
				`	"What happened to the researchers?"`
					goto researcher
				`	"Is it safe to drive over the ice?"`
					goto ice
				`	"Are there sea-monsters beneath the ice?"`
					goto monster
			label researcher
			`	"Long time ago. Researcher convoy look for bacterial pharmaceutical. Many researcher convoy in area. Most smart use native pathfinder, like me. This convoy, they not so smart, they say pathfinder waste of money. They go out in ice alone and they never come back."`
			`	Hjlod shrugs. "Maybe they fall in ice. Maybe they get eaten by Ice-Slug. Maybe avalanche fall on them. Maybe fall on own veapon and die. Nobody ever find them. Tour-guide book say that some night you can see their ghost haunting ice. Tour-guide book dumb."`
			`	"Any more question?"`
				goto questions
			label ice
			`	"Very safe. Except when not. Ice always frozen, but sometime very thin. I can tell. You cannot. Trust Hjlod and you not drown and get eaten by Ice-Slug."`
			` 	"You have more question?"`
				goto questions
			label monster
			`	"Every world have animal. Some animal big, some small. Voolly Octopus, mostly small. Size of this Ice-Crawler, only. Vorst in Glaesisvellir ice-plains is Ice-Slug. They can squeeze through tiniest hole in ice. This take long time, for they are as big as house. They have no teeth, too hard to squeeze through little hole, but they have acid spit. Put hole in Ice-Crawler, easy. They eat many foolish researcher and tourist."`
			`	She smiles at you. "You're not foolish tourist. You are with Hjlod. You are safe."`
			`	"Any more question?"`
				goto questions
			label noquestion
			`	"Is smart, not to ask question. Hjlod tell you best parts only." Hjlod gives you a quick grin and swivels back into her makeshift cockpit.`
			`	"Enough ice-plain, now ve go to next stop."`
			scene "scene/iceplains2"
			choice
				`	(You look out at the bright Ice-Plains as they slowly disappear into the dull horizon.)`
			`	The Ice-Crawler plows through snow, hill, mountain, and tundra for another hour or two before a massive and oddly shaped rock passes into view. Hjlod stops the Ice-Crawler so you can look at it.`
```
This provides a label right before the questions, which you loop back to after asking any question. This means you can't exhaust every option like you could before, but it results in a much cleaner script.

				decline
			label fixme
			`	After hours of tinkering with the guts of the machine you've managed to repair most of it, and all it cost you was one bad burn on your arm. Now, what it comes down to is a sequence of wires. If you can plug these in correctly, the Ice-Crawler should start back up.`
			choice
				`	(White, Green, Blue.)`
					goto boom
				`	(Green, White, Blue.)`
					goto greatsuccess
				`	(Blue, Green, White.)`
					goto boom
			label boom
			` 	You're confident that this is the correct sequence of wires. You plug each one in and start the Ice-Crawler up.`
			`	The vehicle hums to life.`
			choice
				`	(Pump your fist.)`
					goto fist
				`	(Cheer.)`
					goto cheer
			label fist
			` 	In a moment of levity you pump your fist into the air.`
			`	Unfortunately, your fist does not make it very far. As the Ice-Crawler, with its criss-crossed wires, has a critical energy build up that results in an instantaneous completely deadly explosion.`

> 			`	The Ice-Crawler, your entire body, your fist, and more than a couple hundred tons of snow, are all incinerated in a fraction of a second.`
> 			scene "scene/smeer"
> 				die

Reviewer 3: ```suggestion
			scene "scene/smeer"
			`	The Ice-Crawler, your entire body, your fist, and more than a couple hundred tons of snow, are all incinerated in a fraction of a second.`
				die
```
Looks like there's a bug where you can't die right after a `scene`, so we gotta re-arrange it to murder \<first> \<last> appropriately.

			`	Alone now with the Ice-Slug and the pain. You consider briefly your mistakes before you are dissolved and eaten.`
				die
			label help
			`	Damn your pride, "Hjlod help! Something's got my foot!"`
			`	Hjlod looks back at you with a dull expression on her face. "Is it purple-grey in color?"`
			`	"Yes, yes! Quick help. I don't know what to do."`
			`	"Vhat you do, is listen to Hjlod, and not make stompy circles in snow like spoilt child."`
			`	"Please Hjlod!" You cry as the last of your boot dissolves away and you feel the slimy slug on your exposed foot.`
			`	"It is too late. Ice-Slug has you. Vhat you do now is die. Horribly." Hjlod turns and starts jogging away from you.`
			choice
				`	(Try and pull your leg free.)`
					goto free
			label travel
			`	The cold monotonous trudging weighs on you as the hours pass. You're dimly aware of Nifel's two suns as they make their journey across the wide open sky.`
			`	Evening and exhaustion approach hand in hand and as the last sun sets you're not sure how much farther you can go. `
			`	"There!" Hjlod breaks the silence of hours as she points at the weak flicker of light in the twilight landscape.`

> 			`	"This is good." Hjlod admits hoarsely as you both stumble toward the light. "Ve Vould have not survived night."`

Reviewer 3: ```suggestion
			`	"This is good." Hjlod admits hoarsely as you both stumble toward the light. "Ve vould have not survived night."`
```

			`	'For heroic and selfless service to the ideals and prosperity of The Deep.'`
				decline

event "asgard hjlod"

mission "Skadenga Call 4"
	name "Hjlod's goodbye"
	invisible
	destination "Nifel"
	to offer
		has "Skadenga Call 2: done"
		has "event: nifelhjlod"
	on complete
		event "hjlod waits again" 360
		conversation
			`	A familiar woman is waiting for you outside your ship. Although it's been more than seven years since you've last seen her, you recognize her instantly. Her eyes are brighter than when last you parted, and she has the same lopsided grin that you remember.`
			`	"<first>, I saw your ship landing, and thought I vould say hello."`
			`	You ask her how she has been, and she tells you how the small community of Skadenga have rebuilt and thrived here. That there has even been a small amount of immigration to their tiny town. That word had gotten out about their desperate escape from, Asgard.`
			`	She tells you that she forgives you the past, and hopes you find whatever it is you are looking for among the stars. She even hugs you before she leaves.`
			`	As she walks away you wonder, though she might forgive you, if you can ever forgive yourself...`

> 			scene "scene/journey"

Author: I'll just remove it

			`	With that Thorleif and Ragnhild make their way off your ship. Thorleif only takes a few steps out into the quiet Norn spaceport before stopping suddenly and turning back towards you.`
			`	"I almost forgot. The old man, Skaldgar, well he had quite a bit of money saved up. I don't think he hardly spent anything on himself. Anyways, I've decided to split the inheritance with you. After taxes your half comes out to around 500,000 even."`
			`	Thorleif, shrugs, laughs, and walks with Ragnhild hand in hand out into the foggy mists of Norn.`

mission "Home for Skadenga 1"
	name "Home for Skadenga"
	description "Search for the Setstokkr on Windblain and Asgard and then return back to <source>."
	minor
	cargo "Setstokkr" 1
	to offer

> 		has "Deep: Questions:done"

Reviewer 3: ```suggestion
		has "Deep: Questions: done"
```

			`	Finally, you reach the massive Goddess-Stone itself. You see now almost no part of the wide surface is bare. There are images within images, letters within letters. These exotic letters bear no relationship to the runes found on the Setstokkr. These are truly alien. `
			choice
				`	(You've seen enough, and there is much that needs to be done. You head to the shuttle.)` 
					goto shuttles
			label shuttles			
			`	A lone figure stands next to the idling shuttle. As you draw closer you recognize Hjlod.`
			`	"You see!" Hjlod says, "Stick with Hjlod, and you live, probably." She holds up a bandaged three fingered hand and shrugs. "Mostly."`
			`	Hjlod offers one of her lopsided grins and then leaves you alone with the shuttle. It is time to go.`
				accept
	on stopover
		dialog
			`	You land on the final world you were asked to visit. Once you're done here you should return to Nifel.`
	on complete

> 		event "Nifelwaiting" 15

Reviewer 3: ```suggestion
		event "nifel waiting" 15
```
Events can have spaces in them, but should be lowercase by default (which most of yours are, just for this one in particular). The spaces will help make them a bit more readable in the files.

		payment -2000
		conversation
			`	Upon landing you get voice recorded message from Hjlod.`
			`	"Hey <first>, ve're busy evacuating isolated Skadenga communities. Come back in few veeks vhen things settle down."`
			`	The message abruptly ends.`
				decline
				

> event "Nifelwaiting"

Reviewer 3: ```suggestion
event "nifel waiting"
```

			
mission "Home for Skadenga 2"
	name "Home for Skadenga"
	invisible

> 	minor

Reviewer 3: ```suggestion
```

If a mission appears on landing, that gives it a very high priority that doesn't work with `minor`, which is causing CI warnings.

					goto ran
			label relics
			`	"Relics, eh?" Aldfrith cackles. "Sailors will always be over-eager to tell a stranger about fantastic treasures in the deep waves."`
			`	The old man pauses and scratches his head for a moment. "But, alas, I can think of nothing wood-made that would be considered a relic. The salty air and saltier sea don't allow untreated wood to linger long in these parts. Unless its a boat or a home, your relic is not here."`
					goto ran
			label ran
			`	You turn your attention to the great gleaming metal wall in the sea. Unlike the Goddess-Stone of Nifel, this one has no towering figure upon it. Instead it is more abstract, with fractal swirls, and wave like patterns.`
			`	As you stare at it you feel a wildness growing inside your heart, that dissipates the moment you look away. You turn and leave the stone to its wave, and the man to his stone.`
			`	You run your tracker in Hrithfjall once more, but again it fails to lock onto anything. There are faint diffuse traces of the exotic material all around you, but nothing concrete.`
			`	There is little more you can learn here at the moment, and so you head back to your ship.`
				accept
				
	to complete

> 		has "event: Nifelwaiting"

Reviewer 3: ```suggestion
		has "event: nifel waiting"
```

			choice
				`	(You start to answer..)`
			`	Jakob interrupts you before you have a chance to say anything. "Now, listen, you just think it over. We don't need your answer now. All we need is for you to guide those people to Asgard. If you do that, the Transport Union on Valhalla will have the ships you need. If you don't, well, things might be more difficult for everyone. And remember, we'll pay you in real credits. Not charm bracelets."`
			`	"Lars!" Jakob calls, and the administrator and his two guards promptly enter the room. "Show Captain <last> back to their ship. Our meeting is concluded."`
			`	"Of course, sir."`
			`	The guards grab you and 'escort' you off the premises.`
			
		
		
		
mission "Home for Skadenga 3"
	name "Home for Skadenga"
	invisible

> 	minor

Reviewer 3: ```suggestion
```

			`	Most worlds in The Deep have 'Colonial Graffiti', and they are proud reminders of our enterprising colonial origins. They are sometimes known in older texts, and by some outdated communities as God-Stones and Elf-Stones.`
			`	Here at the Helsingor Family-Fun-Shopping-Center we are committed to celebrating our past! Which is why we renovated this old useless colonial structure and made it a vital part of our vibrant community!'` 
			`	You finish reading the plaque, and look at the dull grey metal wall, and the inhuman things carved upon it. It looks nothing like a ship's hull.`
			label ship
			`	There is nothing more you can do here. You head back to your ship.`
				accept
				
	to complete
		has "Home for Skadenga 2: offered"
		
		
			

	
mission "Home for Skadenga 4"
	name "Transport for Skadenga"
	minor
	description "Travel to <destination> to secure transportation for the Skadenga."
	to offer
		has "Home for Skadenga 2: done"

> 		has "event: Nifelwaiting"

Reviewer 3: ```suggestion
		has "event: nifel waiting"
```

				`	"They need to go to Asgard."`
			`	Kaiden beams. "Brilliant. Just brilliant. I was just telling my colleagues here about what a great place Asgard would be for a gaggle of misfit primitives. The publicity will be huge. The universities will probably want to run various studies. There will be all sorts of government programs and initiatives there to help them accept the proper Deep culture, and become productive citizens like the rest of us."`
			`	Grayden, Braiden and Kaiden all start shaking each others hands and congratulating each other on being part of something so inspirational.`
			`	After their celebration is over Kaiden turns back to you. "Our transport fleet is at your disposal. They are being refitted at Mani in the Avior system."`
			`	The three stand up in unison. Grayden and Braiden leave and Kaiden turns from the door. "We are so happy you stopped by today and gave us this opportunity to really give back to the needy communities out in our neighborhood."`
			`	With that they are all gone, and your escort returns to lead you out.`
			
		
			
mission "Home for Skadenga 5"
	name "Transport for Skadenga"

> 	minor

Reviewer 3: ```suggestion
```

	landing
	description "Travel to <destination>, and learn about the Transport Union Fleet"
	to offer
		has	"Asgard"
	source "Valhalla"
	destination "Mani"
	on complete

> 		event "manirefit" 21

Reviewer 3: ```suggestion
		event "mani refit" 21
```

		conversation
			apply
				set "Asgard2"
			`	Mani is a quiet empty moon, with only a few hubs of human activity. The Deep Secruity forces, Transport Union, and other semi-governmental fleets are all serviced here.`
			`	A Transport Union executive meets you when you touch down.`
			`	"It'll take a while for the ships to be ready, Captain <last>. We're primitive-proofing them so that those ice-savages from Nifel don't accidently kill themselves."`
			`	He goes over some time-tables, "I'll send you a message when the ships are ready, it should take about three weeks.` 
			`	He shakes his head. "When the ships return covered in hay, feces and other low-tech trash, it'll take us weeks to get them in a usable state again."`
			scene "scene/citydark"
			
			
			

> event "manirefit"

Reviewer 3: ```suggestion
event "mani refit"
```

			
			
mission "Mani Refit"
	name "Mani Refit"
	to offer

> 		has "event: manirefit"

Reviewer 3: ```suggestion
		has "event: mani refit"
```

			
			
mission "Home for Skadenga 7"
	landing
	name "Evacuate <origin>"
	description `Escort a fleet of freighters, refitted for carrying primitives, to <destination>.`
	source "Nifel"
	destination "Asgard"
	to offer
		has "Home for Skadenga 6: done"
	on offer
		dialog
			`The fleet touches down just outside the over-crowded village of Udainsakr. The landscape is filled with the gathered Skadenga.`
			`	The Ondurdis approaches you. "Leaving is difficult. But all winters must come to end."`
			`	She watches as her people board the ships. "I must rejoin my folk. But thank you for this, and we shall speak again at our new home."`
	
	npc accompany save
		government "Deep Security"
		personality timid escort
		ship "Bactrian" "DTU Endeavor"
		ship "Bulk Freighter" "DTU Consistency"
		ship "Bulk Freighter" "DTU Dependable"
		ship "Freighter" "DTU Empathy"
	on visit
		dialog
			`You have reached <planet>, but the Transport Union ships have not all arrived yet. You should take off and wait for them to catch up with you.`
			
	on complete
		payment 850000

> 		event "Skadengadeath" 367

Reviewer 3: ```suggestion
		event "skadenga death" 367
```

			scene "scene/buildings"
			`	"These folk are lost." She says. "We are lost."`
			`	She looks at you with her tear-wrecked face and says,"You are instrument of Skade. I saw it. But why? Why this? Why to us?"`
			`	After that she will say no more to you, she lifts her eyes to the sky and begins to chant wildly. A few Skadenga who have made it this far join her. Various news outlets swarm around them to record the Ondurdis' breakdown.`
			`	You back away slowly, and bump into someone. You turn to see the real-estate agent Carla Von Hausen smiling at you.`
			`	"Well done, Captain <last>. I've been authorized to tell you that my superiors have deposited 850,000 credits into your account, as was agreed upon earlier."`
			choice
				`	"Your superiors??"`
			`	Carla grins. "The Census and Culture Bureau. I am one of their field agents. They are most pleased with how you handled this. The Skadenga have been a problem for centuries, and you solved it."`
			`	She pats you on the back. "Anyways, I just wanted to pass that on. I have lots of work to do, thanks to you." She winks and wanders into the crowd.`
			`	You return to your ship to count your money.`
			
			
			

> event "Skadengadeath"

Reviewer 3: ```suggestion
event "skadenga death"
```

	
	
mission "Skadenga Call 1"
	landing
	name "Check on Hjlod"
	description `Fly to <destination>, and see what Hjlod and the Skadenga need from you.`
	source
		government "Republic"
	destination "Asgard"
	to offer
		has "Home for Skadenga 7: done"

> 		has "event: Skadengadeath" 

Reviewer 3: ```suggestion
		has "event: skadenga death" 
```

			`	Carla gives you a tight smile. "Decided to do the right thing in the end? We appreciate it."`
			`	The police officers enter your ship and start restraining and removing the the Skadenga aboard it.`
			`	"As always, we appreciate your help and value our continued working relationship. Expect a payment in the next couple days, after I've gotten it approved, for your aid." Carla steps back to assess the Skadenga, most of which are sullen and reserved. One though, is screaming, kicking and biting.`
			`	The unruly Skadenga is dragged past you, and spit splashes against your cheek. "I'll kill you, <first>. If ever ve meet again I'll kill you. You've killed me. You've killed me." Hjlod's red and tear-stained face recedes as officers drag her away.`
			`	"Savages." Carla says with disgust. "We'll take it from here. Have a wonderful day, Captain <last>.`
				decline
			label takeoff
			`	You punch it, and your ship burns for the stars. Hjlod visibly relaxes when the thrust kicks in. She closes her tired eyes and slumps to the floor.`
			`	"Thank Skade", is all she says.`
				accept
	on complete
		payment 12481

> 		event "thebook" 200

Reviewer 3: ```suggestion
		event "the book" 200
```

> 		event "nifelhjlod" 2700

Reviewer 3: ```suggestion
		event "nifel hjlod" 2700
```

		conversation
			` 	Your ship touches down just outside the abandoned village of Udainskar. Silently your passengers disembark and drift into the world they once knew and thought never to see again. The village appears to have weathered the abandonment well, and it appears if the people here will not have too much trouble picking up where they left off.`
			`	Hjlod lingers besides your ship for just a moment. "The credits are in hard-drive on your ship's console. I hope it makes you happy."`
			`	"I do not know vhy you sent us to that place. But I am happy to be home. So go, and maybe one day I vill not hate to see you again."`
			scene "scene/snowvillage"
			

> event "nifelhjlod"

Reviewer 3: ```suggestion
event "nifel hjlod"
```

> event "thebook"

Reviewer 3: ```suggestion
event "the book"
```

			
mission "Skadenga Call 3"
	name "Blood Money"
	landing
	source
		government "Republic"
	destination
		government "Republic"
	invisible
	to offer
		has "Skadenga Call 2: declined"
	on offer
		payment 300000

> 		event "asgardhjlod" 2000

Reviewer 3: ```suggestion
		event "asgard hjlod" 2000
```

		conversation
			`	You receive a message from your bank that 300,000 credits have been deposited into your account from the Census and Culture Bureau of The Deep. The following message is attached:`
			`	'For heroic and selfless service to the ideals and prosperity of The Deep.'`
				decline
		
			

> event "asgardhjlod"

Reviewer 3: ```suggestion
event "asgard hjlod"
```

			
mission "Skadenga Call 4"
	name "Hjlod's goodbye"
	invisible
	destination "Nifel"
	to offer
		has "Skadenga Call 2: done"
		has "event: nifelhjlod"
	on complete

> 		event "Hjlodwaitsagain" 360

Reviewer 3: ```suggestion
		event "hjlod waits again" 360
```

		conversation
			`	A familiar woman is waiting for you outside your ship. Although it's been more than seven years since you've last seen her, you recognize her instantly. Her eyes are brighter than when last you parted, and she has the same lopsided grin that you remember.`
			`	"<first>, I saw your ship landing, and thought I vould say hello."`
			`	You ask her how she has been, and she tells you how the small community of Skadenga have rebuilt and thrived here. That there has even been a small amount of immigration to their tiny town. That word had gotten out about their desperate escape from, Asgard.`
			`	She tells you that she forgives you the past, and hopes you find whatever it is you are looking for among the stars. She even hugs you before she leaves.`
			`	As she walks away you wonder if you can ever forgive yourself.`
			scene "scene/journey"
			

> event "Hjlodwaitsagain"

Reviewer 3: ```suggestion
event "hjlod waits again"
```

			
mission "Skadenga Call 5"
	name "Hjlod's worse goodbye"
	invisible
	source
		government "Republic"
	destination
		government "Republic"
	to offer
		has "Skadenga Call 3: declined"

> 		has "event: asgardhjlod"

Reviewer 3: ```suggestion
		has "event: asgard hjlod"
```

	on offer

> 		event "asgardhjlodletter" 1000

Reviewer 3: ```suggestion
		event "asgard hjlod letter" 1000
```

		payment 27
		dialog
			`	You receive a message from the Census and Culture Bureau of The Deep. The Deep Citizen known as Hjlod has died. Cause of death appears to be related to heavy substance abuse. The Bureau has determined that you are her closest relation and has forwarded you her entire inheritance of: 27 credits.`

> event "asgardhjlodletter"

Reviewer 3: ```suggestion
event "asgard hjlod letter"
```

			`	You ask him what this is all about.`
			`	"I don't agree with what the Union, or The Deep are doing. I was raised in a small town where we still respected our past, and I couldn't just sit around and do nothing. So listen, with the Union against you there's no transport company that will pick up your contract, with one exception. There's an old renegade mineral baron on Muspel. He hates the Union, and may be willing to do what you ask."`
			choice
				`	"How do I find him?"`
			`	The Union employee begins moving away from you. "Name's Hroar, you can usually find him in the Red Rock Saloon, inside the Muspel Spaceport."`
			`	The disgruntled Union employee then says, quite loudly, "No ser! I do not know where the bracelet shop is, please stop asking me!" And hurries away from you.`
				accept
	on complete
		dialog
			`	You've arrived at the rusty hued world that is Muspel. You should head into the spaceport to see if you can find Hroar.`

mission "Home for Skadenga 9"
	name "Home for Skadenga"
	minor
	invisible
	to offer
		has "Home for Skadenga 8: done"
	source "Muspel"
	destination
		government "Republic" "Neutral" "Syndicate" "Free Worlds" "Deep"
	on offer

> 		event "waitforit" 13

Reviewer 3: ```suggestion
		event "hroar message" 13
```
A slightly more descriptive event message

			`	He grins. "So, what do you say? Deal?"`
			choice
				`	"Deal."`
					goto deal
				`	"No deal."`
					goto lastchance
			label lastchance
			`	Hroar leans back. "Kid, listen up. If the Union won't do it, and I won't do it. Then no one will. You'll either have to get those 2,200 bunks yourself, or it just ain't happenin'. I promise I won't ask anything of you too illegal. So, you sure you want to say no?"`
			choice
				`	"I am in. Deal."`
					goto deal
				`	"I am out. No deal."`
					goto nodeal	
			label nodeal
			apply
				set "refusedhroar"
			`	Hroar smirks. "Alright kid. You're out. Get outta my booth, and good luck on your little charity-adventure."`
			`	Hroar goes back to drinking his whiskey as the Saloon's swinging door hits you on the butt on your way out."`
				decline
			label deal
			apply
				set "dealwithhroar"
			`	"Good. Good. People making deals. That's what society is all about, ya know? Alright, listen up, it'll take me about two weeks to get my fleet ready to transport your friends. So, come back here then and we'll get this show on the road."`
				decline

> event "waitforit"

Reviewer 3: ```suggestion
event "hroar message"
```

		
		
mission "Home for Skadenga 10"
	name "Head Back to Muspel"
	landing
	to offer

> 		has "event: waitforit"

Reviewer 3: ```suggestion
		has "event: hroar message"
```

	source "Nifel"
	destination "Windblain"
	to offer
		has "Home for Skadenga 11: done"
	on offer
		dialog
			`The fleet touches down just outside the crowded village of Udainsakr. The landscape is filled with the gathered Skadenga.`
			`	The Ondurdis approaches you. "Leaving is difficult. But all winters must come to end."`
			`	She watches as her people climb aboard the fleet. "I must rejoin my folk. But thank you for this, and we shall speak again at our new home."`
			`	She leaves you with a warm smile in the wintry village.`
				accept
	
	npc accompany save
		government "Merchant"
		personality timid escort
		ship "Behemoth" "HHH Hroar's Howl"
		ship "Hauler" "HHH Red Dawn"
		ship "Hauler" "HHH Crimson Carrier"
		ship "Hauler III" "HHH Unstoppable Force"
		ship "Hauler II" "HHH Fixed Income"
		ship "Mule" "HHH Breadbasket"
		ship "Star Queen" "HHH Land of Plenty"
		ship "Star Barge" "HHH Bandwagon"
		ship "Bounder" "HHH Rustborn"
	on visit
		dialog
			`You have reached <planet>, but the Hardy Hroar's Hauler ships have not all arrived yet. You should take off and wait for them to catch up with you.`	
	on complete

> 		event "Skadenga life" 367

Reviewer 3: ```suggestion
		event "skadenga life" 367
```

			`	"So, you are the ice that falls from the stars?" Asks Aldfrith with a hint of mirth in his eyes.`
			`	We are Skadenga, daughters of Skade. But, I think now, she sends to her sister Ran." The Ondurdis replies.`
			`	Aldfrith nods slowly. "We are the folk of Ran; the offspring of the surf, the students of the wind, and the watchers of the storm. We welcome you, and your people here. Though there are few here at present, many more of our number can be found out amongst the surging seas."`
			`	Aldfrith glances at the growing number of Skadenga. "If you follow me, I will escort you to our village, and show you what buildings you may use. Though I fear we can house so many."`
			`	The Ondurdis smiles. "We thank you for your hospitality. It is good to find others who remember the old ways. Fear not, though, we are resourceful lot."`
			`	The Ondurdis turns to you. "Meet me near the ships, at makeshift spaceport in few hours. I have much to do at moment." She, and many others, then follow Aldfrith and his group to the halls of Hrithfjall.`
			scene "scene/seasidecliffs"
			

> event "Skadenga life"

Reviewer 3: ```suggestion
event "skadenga life"
```

			`	The leader of the Skadenga returns to her people on the windswept hills of the small picturesque town of Hrithfjall.`
			label done
			`	With the conversation concluded you wander back toward your ship. You hear the hustle and bustle all around you of eager people pursuing a new and exciting life. The promise of a new beginning is almost as intoxicating as the faint hint of sea-salt on the cool breeze that ruffles your jacket.` 
			`	You wonder if you're forgetting anything. If you should find Hroar, or Hjlod and speak with them. But they are nowhere in sight, and you realize there will be time to see them again.`
			`	You climb aboard your ship and leave these people to their new home, and you wonder, in all the bright stars and warm worlds, where might your home be?`
			scene "scene/sunset"
				decline

mission "Homecoming to Skadenga"
	description "Take the mourning Skadenga to <destination>, so that they can enact the rites of their dead."
	source "Windblain"
	destination "Nifel"
	passengers 5
	to offer
		has "Home for Skadenga 13: declined"

> 		has "event: Skadenga life"

Reviewer 3: ```suggestion
		has "event: skadenga life"
```

			`	She then nods at you. "If you could transport  family members of  deceased to return their stones. It would mean much to everyone here, and we would pay you what we could."`
			choice
				`	"Of course."`
					goto yes
				`	"I'll do it for free."`
					goto yesyes
				`	"No thanks."`
			`	You explain to them you have more pressing matters to attend to, and cannot be a taxi service for their grieving. They graciously accept your excuses and you leave back to your ship.`
				decline
			label yesyes
			`	The Ondurdis shakes her head. "You have already done too much for us for too little pay. We can afford to pay you, and we will."`
			label yes
			`	"The five Stone-Bearers will accompany you back to your ship. May Skade and Ran watch over your journey." The Ondurdis turns this into a toast, which everyone gladly participates in.`
			`	Hjlod walks with you, and your new passengers, back to your ship. "Be well, <first>."`
				accept
	on complete
		payment 50000

> 		event "hjlodwaits" 360

Reviewer 3: ```suggestion
		event "hjlod waits" 360
```

		dialog
			`	The abandoned village of Udainsakr is cold, dark and half-buried in un-retreated snow. Everything feels colder than you remember and you wonder how long before this planet is a perpetual snowball.`
			`	The Five Skadenga you brought are polite and circumspect. Upon landing they leave your ship and head for the Goddess-Stone of Skade, with its attendant piles of stones. You wait for them in the spaceport.`
			

> event "hjlodwaits"

Reviewer 3: ```suggestion
event "hjlod waits"
```

			
mission "Stones of Skadenga 1"
	name "Sones of Skadenga"
	source "Nifel"
	passengers 5
	destination "Windblain"
	description "Return the Skadenga mourners to <destination>."
	to offer
		has "Homecoming to Skadenga: done"
	on offer
		conversation
			`	The Skadenga each navigate their way to a specific cairn and stand there for a moment, speaking or chanting, and then placing their stone upon it. Solemnly they return to you with their task complete. There is nothing else for them to do here and they request you take them home.`
			choice
				`	(Return the mourning Skadenga to Windblain.)`
					goto home
				`	(Leave them here.)`
			`	You let them know that you only agreed to take them here, not bring them back. You shut your hatch and leave them to fend for themselves.`
				decline
			label home
			`	They thank you, and board your ship.`
				accept
			
				
	on complete

> 		event "skadengamourners" 90

Reviewer 3: ```suggestion
		event "skadenga mourners" 90
```

		dialog
			`	The Skadenga are gladened to be home. Something about being on Windlbain lifts their spirits and they thank you for your help.`

> event "skadengamourners"

Reviewer 3: ```suggestion
event "skadenga mourners"
```

	
mission "Stones of Skadenga 2"
	name "Stones of Skadenga"
	description "Take the mourning Skadenga to <destination>, so that they can enact the rites of their dead."
	source "Windblain"
	destination "Nifel"
	passengers 8
	landing
	to offer

> 		has "event: skadengamourners"

Reviewer 3: ```suggestion
		has "event: skadenga mourners"
```

	job
	repeat
	source "Windblain"
	destination "Nifel"
	passengers 1 10 .9
	description "Take the mourning Skadenga to <destination>, so that they can enact the rites of their dead."
	to offer
		has "Stones of Skadenga 4: declined"
		random < 30
	on offer
		dialog
			`	The mourning Skadenga board your ship.`
	on visit
		dialog phrase "generic passenger on visit"
		
	on complete
		payment
		dialog
			` The mourners take their small stones and head off to visit the cairns of their ancestors.`

mission "Stone-Bearers: To Windblain"
	job
	repeat
	source "Nifel"
	destination "Windblain"
	passengers 1 10 .9
	description "Return the Skadenga Stone-Bearers to <destination>."
	to offer
		has "Stone Skadenga: To Nifel: done"
		random < 30
	on offer
		dialog
			` The Skadenga mourners embark on your ship. Happy to see the ice-ball of Nifel left behind.`
	on visit
		dialog phrase "generic passenger on visit"
	on complete
		payment
		dialog
			`The Skadenga disembark and return to their lives on Windblain.`
			

mission "Hjlod RemembersWindblain"
	name "Hjlod Remembers"
	source
		government "Republic"
	destination "Windblain"
	to offer 
		has "Homecoming to Skadenga: done"

> 		has "event: hjlodwaits"

Reviewer 3: ```suggestion
		has "event: hjlod waits"
```

		conversation
			`	Hjlod is waiting for you when you land. She looks healthy, strong, and happy. Less gaunt and stressed then you recall her looking on Nifel when you first met.`
			`	She gives you a hug. "Alvays good to see you, <first>. I have been thinking, vith all that vent on and needed to be done. I never had chance to thank you for saving me from cold."`
			`	She reaches into her pocket and pulls out a small pouch. "I have stone that has long been in my family. I wish to give it to you. Keep you cold in hot ship, remind you of Hjlod in far off galaxy."`
			`	She gives you a glowing blue Tear of Skade, just like the one The Ondurdis gave you sometime before.`
			scene "outfit/skadetear"
			`	"How many of these are there you ask?"`
			`	"Only few", she says, "They were found around the Goddess-Stone in the early days. Passed from Skadenga mother to Skadenga daughter. I have no daughter, but I have you, so this is now yours."`
			`	You thank her, and spend the rest of the day with her, talking, laughing and walking upon the friendly earth of Windblain.`
	
mission "Hjlod RemembersNifel"
	name "Hjlod Remembers"
	source
		government "Republic"
	destination "Nifel"
	to offer 

> 		has "Skadenga Call 4:done"
> 		has "event: Hjlodwaitsagain"

Reviewer 3: ```suggestion
		has "Skadenga Call 4: done"
		has "event: hjlod waits again"
```

			`	Hjlod finds you and gives you a small smile. "I am glad you came."`
			`	She reaches into her pocket and pulls out a small wooden box, similar to the one you saw displayed at Asgard all those years ago.`
			`	"I have been thinking", she says, "Maybe all this vas vill of Skade. Maybe you took us to Asgard to make us stronger, to make us better. It is bitter lesson, but I understand it now."`
			` 	She holds out the box. "This is true of Skakenga. Only few ever found, and this I give to you. For saving my life, for bringing us back here, for making us stronger."`
			`	The box opens displaying a small blue glowing rock. "This is Tear of Skade. alvays makes cold. Alvays. Useful on hot ship."`
			scene "outfit/skadetear"
			`	She hands it to you. "Keep this vith you, and know Skadenga vith you too."`	
			`	Her eyes water and she wipes them. "Your story is forever chiseled on our stones."`
			`	She turns and leaves you there, holding a Tear of Skade, and shivering in Nifel's unforgiving cold.`
			
mission "The Book of Skadenga"
	invisible
	source
		government "Republic"
	destination
		government "Republic"
	to offer
		random < 10
		has "Skadenga Call 2: done"

> 		has "event: thebook"

Reviewer 3: ```suggestion
		has "event: the book"
```

			`	There's an old-style book there, bound up in a ice-blue cover titled, "The Skakdenga Saga: Against All Odds by Hjlod Hrimnirsdottir"`
			choice
				`	(Ignore the book and move on.)`
					goto ignore
				`	(Pick the book up.)`
			`	The book feels solid in your hands, and you scan through it. There are chapters about near death survival on Nifel, corrupt Deep officials, the colorful haulers on Muspel, and the ultimate triumph on Windblain.`
			`	You place the book back on its' stand.`
			`	The attendant notices you. "It's pretty popular, that one. I hear it started this whole 'back-to-nature' primalism cult."`			
			`	He shrugs. "Whatever sells, ya know. Anyways, you buying?"`
			`	You decline. You already know that story.`
				decline
			label ignore
			`	You keep walking. You already know that story.`
				decline
		
	
mission "The Angry Letter"
	invisible
	source
		government "Republic"
	destination
		government "Republic"
	to offer
		or
			has "Skadenga Call 5: done"
			has "Skadenga Call 5: offered"
			has "Sakdenga call 5: declined"
		random < 10

> 		has "event: asgardhjlodletter"

Reviewer 3: ```suggestion
		has "event: asgard hjlod letter"
```

			`	Hjlod turns back to her controls and gets the Ice-Crawler moving forward again. "That is enough about stupid rock and stupider man."`
			`	"Now ve go to river."`
					goto riverlike
			label byerock
			`	"Now ve go to river."`
					goto riverlike
			label riverlike
			choice
				`	(The large oddly shaped rock falls out of your field of view as the Ice-Crawler turns away from it.)`
			scene "scene/snowfield"
					goto river
			label river
			`	The uneventful journey to the boiling ice river of Slidr is suddenly interrupted when the Ice-Crawler begins to shake wildly for a few moments, followed by a thunderous boom from somewhere outside.`
			`	After Hjlod regains control of the crawler, she immediately veers away from the noise. "That vas Nifel-quake, and loud noise was avalanche. This is not good."`
			`	Despite her calm demeanor she wields the Ice-Crawler with reckless abandon as she tries to outrun the rumbling snow. The machine protests with creaks and whirrs and pops as it goes faster than it has gone for hundreds of years.`
			`	"I think..." Hjlod starts to say when a mountain of snow crashes into the Ice-Crawler causing it to roll and tumble for over a kilometer.`

> 			`	You and Hjlod hold tight to your seat-belts as you're tossed around. Eventually, the crawler rolls to its side and comes to a rest.`

Author: Obviously she can't outrun an avalanche, but that doesn't mean she wouldn't try to outrun it. I think that is a pretty standard human response. 

Per your suggestion I've edited the wording around the tumble and kilometer stuff to make it sound less damaging.

> 			`	You and Hjlod hold tight to your seat-belts as you're tossed around. Eventually, the crawler rolls to its side and comes to a rest.`

Reviewer 4: It's a standard human response, it's also an incredibly stupid one and the very first thing that any sort of wilderness survival experience would teach you not to do.

If you lived in this environment it would be drilled into you to get out of the way, to find shelter, to do anything except remain in the path of the avalanche. She's a native, she should know better.

> 			`	You and Hjlod hold tight to your seat-belts as you're tossed around. Eventually, the crawler rolls to its side and comes to a rest.`

Author: I've changed "outrun" to "avoid". 
I think that should satisfy your concerns.

			choice
				`	(You've seen enough, and there is much that needs to be done. You head to the shuttle.)` 
					goto shuttle
			label shuttle
			
			`	A lone figure stands next to the idling shuttle. As you draw closer you recognize Hjlod.` 
			`	"I just vant to thank you." Hjlod's grey-blue eyes glitter in the morning light. "I alvays say Hjlod keep tourist safe. But yesterday you save Hjlod. I vill not forgot, know this."`
			`	Hjlod offers one of her lopsided grins and then leaves you alone with the shuttle. It is time to go.`
				accept
			
				
			label jacket
			`	You throw on the jacket and stuff a few rags in your coat for good measure.` 
			`	Hjlod nods at you and opens the hatch, "It is only meter to surface."`
			` 	She plunges into the snow and swim-climbs her way up it. You follow her, and though the cold is shocking you are relieved when you quickly rise above it into the icy clear air above.`
			`	"Stay close," she says as she begins to trek through the snow, "any path but mine is dangerous."`
			scene "scene/snowplain"
			choice
				`	(Walk in Hjlod's footsteps.)`
					goto travel

> 				`	(Meander around because you're too proud to do what she says.)`

Author: How would you word it?

> 				`	(Meander around because you're too proud to do what she says.)`

Reviewer 4: I call upon @W1zrad for opinions.

> 				`	(Meander around because you're too proud to do what she says.)`

Reviewer: ```suggestion
				`	(Meander around.)`
```
Personally, I'd do this. The second half seems to impose a mood on the player character _way_ too much compared to ES standards. If you wanted to do something that preserves the tone, consider changing it to dialogue so the player feels like they have more of a choice in what they say. Something like this:
```suggestion
				`	"No. I'll walk where I want to."`
```
Even then it feels incredibly... abrasive?

			`	"It is too late. Ice-Slug has you. Vhat you do now is die. Horribly." Hjlod turns and starts jogging away from you.`
			choice
				`	(Try and pull your leg free.)`
					goto free
			label travel
			`	The cold monotonous trudging weighs on you as the hours pass. You're dimly aware of Nifel's two suns as they make their journey across the wide open sky.`
			`	Evening and exhaustion approach hand in hand and as the last sun sets you're not sure how much farther you can go. `
			`	"There!" Hjlod breaks the silence of hours as she points at the weak flicker of light in the twilight landscape.`
			`	"This is good." Hjlod admits hoarsely as you both stumble toward the light. "Ve Vould have not survived night."`
			` 	The night air grows too cold. You slow as your body begins to shutdown and frostbite gnaws at your hands, feet and exposed arm. The light is still too distant, and you can barely move. You think you see the shadows of humanoid figures against the light. But then you stumble and fall. You crawl, for as long as you can, you crawl. And then nothing.`
			choice

> 				`	(In the darkness you feel hands, and then warmth, and then more nothing.)`

Author: That's easy enough to add. But why impower the player to make a choice on something that is inherently not a choice? Passing out it happening to them, they are not choosing to pass out. And no one, in these circumstances, chooses how to pass out.

> 				`	(In the darkness you feel hands, and then warmth, and then more nothing.)`

Reviewer 4: I think it's just that it's a novel, and quite fun, but very unusual use of the choice functionality. Another place that @W1zrad may have more insight than I.

> 				`	(In the darkness you feel hands, and then warmth, and then more nothing.)`

Reviewer: The primary problem is that this isn't in the imperative. Below would be the imperative form:
```suggestion
				`	(Feel hands in the darkness, and then warmth, and then more nothing.)`
```
But that really doesn't make sense. I'd personally say remove the choice altogether and just work it into the normal text if you want to do something that sounds cool like this. I know you're not a fan of walls of text but this is a spot where there's not really a way to get around it...
...unless you switch some stuff around. Maybe work `In the darkness you feel hands, and then warmth, and then more nothing.` into the dialogue, _then_ have `(Wake up.)` or something as your choice.

> 				`	(In the darkness you feel hands, and then warmth, and then more nothing.)`

Author: I have removed this part and just replaced it with the option to (Wake up.)

					goto village
			label village
			`	The pale morning light of the first sun filters in through a synthetic window and onto your face.`
			`	Your body aches and you try to sink back into sleep, but something feels off and you focus on your surroundings.`
			`	You're in a half-wooden, half-synthetic barracks. A strange mix of primitive earth-tech and old colonial architecture.`
			`	That, in of itself, isn't that alarming. What is, however, is the group of people gathered around your bed quietly staring and pointing at you.`
			choice
				`	"Excuse me?"`

> 			`	The crowd steps back and an elderly woman steps forward. Her grey hair is rolled up into tight elegant braids, and her face and hands are covered in faded blue tattoos.`

Author: It's this way because its the only way I know how to do it. This is my 2nd PR and I don't have a strong familiarity with the formatting stuff. I can't make act on your suggestion here, because I don't know how to atm.

> 			`	The crowd steps back and an elderly woman steps forward. Her grey hair is rolled up into tight elegant braids, and her face and hands are covered in faded blue tattoos.`

Reviewer 4: I understand, if I get some time I'll maybe see if I can work it out.

> 			`	The crowd steps back and an elderly woman steps forward. Her grey hair is rolled up into tight elegant braids, and her face and hands are covered in faded blue tattoos.`

Author: Terin has implemented the looped dialog options. I think this is now resolved.

> 			`	The crowd steps back and an elderly woman steps forward. Her grey hair is rolled up into tight elegant braids, and her face and hands are covered in faded blue tattoos.`

Reviewer 4: > Thanks to Terin, I think both issues have now been resolved.

Terin dealt with the looping conversation options, but there's still at least one major block repeated.

> 			`	The crowd steps back and an elderly woman steps forward. Her grey hair is rolled up into tight elegant braids, and her face and hands are covered in faded blue tattoos.`

Author: Which block? There are some subtle differences between a few of the options depending on how you get to the village, if that is what you're thinking.

> 			`	The crowd steps back and an elderly woman steps forward. Her grey hair is rolled up into tight elegant braids, and her face and hands are covered in faded blue tattoos.`

Reviewer 3: Yeah, I don't think there's enough there to break out into separate blocks, unless we REALLY go line by line.

					goto fancypants
					
			label pass
			`	As you try to get around the administrator, his goons grab you. "Respectfully, Captain, this is not an invitation you are at liberty to dismiss."`
					goto fancypants
			label someone
			`	"Ah yes, don't worry, we have the port authority keeping your hill-witch friend busy. You'll likely be done before she is." The administrator nods to his guards who close in on you.`
			`	"Regrettably, this is not an invitation you are at liberty to ignore, so let us be on our way, shall we?"`
					goto fancypants
			label fancypants
			`	You are brought to large research complex just outside the spaceport, and dispoisted in a high-end white minimalistic conference room, with two well-dressed and important looking men.`
			`	"Thank you, Lars. Take the guards and wait for us outside." the man seated across from you says.`
			`	"Of course, sir." Lars turns and is followed outside by his two goons.`
			`	The man who spoke looks at you with a friendly open smile, and says unironically. "I am so glad you agreed to meet with us. I realize an accomplished Captain like yourself is in great demand, so we will endeavor to keep this brief."`

> 			`	"I am Jakob Forsyth, Vice Planetary-Governor of Nifel and CEO of Nifel-Research Solutions Inc., and the man sitting beside me is Henrik Fogel, Deputy Lieutenant-Sub-Director of the Census and Culture Bureau for The Deep."`

Author: Within the mission, the guy who takes credit for curing the Rigellian Plague is also a Foryth. The player can draw connection between him and current Vice-Governor. Outside of the mission, if some else has this name.. that would just be a wild accident. And I would change it here as to not confuse the player.

> 			`	"I am Jakob Forsyth, Vice Planetary-Governor of Nifel and CEO of Nifel-Research Solutions Inc., and the man sitting beside me is Henrik Fogel, Deputy Lieutenant-Sub-Director of the Census and Culture Bureau for The Deep."`

Reviewer 4: I worked it out; Henry Forsyth has been an assistant director or second unit director for 42 different motion pictures, many of which I have seen, and Jakob Forsyth is also a fallout 4 character.

Forsyth is just an unusual enough name I was picking up on it.

> 			`	"I am Jakob Forsyth, Vice Planetary-Governor of Nifel and CEO of Nifel-Research Solutions Inc., and the man sitting beside me is Henrik Fogel, Deputy Lieutenant-Sub-Director of the Census and Culture Bureau for The Deep."`

Author: So, am I good to leave the name in?

> 			`	"I am Jakob Forsyth, Vice Planetary-Governor of Nifel and CEO of Nifel-Research Solutions Inc., and the man sitting beside me is Henrik Fogel, Deputy Lieutenant-Sub-Director of the Census and Culture Bureau for The Deep."`

Reviewer 4: If you don't mind them being names with moderate recognition value from elsewhere then yeah you're good.

			`	"The Deep would never forcibly resettle a population. But with your help we have an opportunity to guide them to a place where they could receive the help they need."`
			`	"We know you recently visited Windblain and Asgard. Sending them to Windblain, a barely inhabited world with no useful public services, would be a huge mistake. These poor wretches disappear into the wild cracks there. Asgard, however, has a robust civil service and safety net that would be perfect to help them be absorbed into our modern culture."`
			`	"This is why I asked the Vice Planetary-Governor to hold this meeting. The Deep needs you to convince these poor backwards people, that they should live on Asgard."`
			`	Henrik shrugs. "For all we know, you've already decided to tell them Asgard is the best choice. In which case we're really not asking you to do anything you weren't already planning on doing. We will also help facilitate the move by providing transportation. Furthermore, for your support in this delicate matter, we will transfer 850,000 credits to your account upon their arrival in Asgard. Really, everyone wins this way."`
			choice

> 				`	(You start to answer..)`

Author: Alright, I have added three beginning responses here.

					goto plaque
				`	(Return to your ship.)`
					goto ship
			label plaque 
			`	The plaque labels the God-Stone as 'Colonial Graffiti' and reads:`
			`	During the Colonization of The Deep in the 26th Century, the earliest colonists  erected large metal slabs and wrote nonsensical pseudo-letters on them. It is believed that this was done in order to raise the prestige of their early worlds. They would claim that they were powerful alien artifacts, or that they were made by an earlier population of humans who had arrived on the world thanks to the intercession of fairies.`
			`	Wanting to profit from superstitious nonsense, many local communities promoted these outrageous ideas about the origins of these structures. Scientists have determined that these structures were once large segments of the metal hulls of the original colony ships. Although, to date, there has been some trouble identifying exactly what kind of metal they were made from.`
			`	Most worlds in The Deep have 'Colonial Graffiti', and they are proud reminders of our enterprising colonial origins. They are sometimes known in older texts, and by some outdated communities as God-Stones and Elf-Stones.`

> 			`	Here at the Helsingor Family-Fun-Shopping-Center we are committed to celebrating our past! Which is why we renovated this old useless colonial structure and made it a vital part of our vibrant community!'` 

Author: My opinion here is there has to be an 'official' story.

 The current Deep missions have the Elves bringing humans to The Deep story as somewhat of a reveal. That means it isn't common knowledge, or if it is, it is treated as a folk-legend and not a serious one. In which case there has to be a serious official story. 

So, what we're seeing here is not any real departure from what is already established by the lore, we're just seeing how the 'official story' rationalizes elements of the 'fairy-tale'.

> 			`	Here at the Helsingor Family-Fun-Shopping-Center we are committed to celebrating our past! Which is why we renovated this old useless colonial structure and made it a vital part of our vibrant community!'` 

Reviewer 4: Right, but this is not just the official story rationalising it. Particularly with the commentary from the shopping centre, the presentation in this context is actively denigrating it. There is a difference.

When there's something you don't want people to look at too closely, you try and avoid drawing attention to it. Effectively shouting, "This is not important!" from the rooftops doesn't do that.

Obviously the official line says, "We likely came here in the 26th Century, and the rest are just folktales."

But to draw from your example, this particular sign would be like putting up a sign in a deeply conservative christian area that says, "The bible is a silly book written by a lunatic 2000 years ago. This is why we've taken this cross and hung advertisements from it! Obviously using the cross for profit-making by non-christian businesses is still respectfully including our past in our current community!"
Can you imagine the outcry, the protests, and the vandalism that would occur in that case today?

It's wildly, and unnecessarily, hostile towards the narrative.

I would actually expect a very long plaque that spends like 1000 words talking about the mythos, and how it waned in significance on Asgard, and what the official line is, and what efforts were made to try and prove the mythos before ultimately failing to find evidence, etc. etc. 
Like, if there's a plaque should be boring. It should work to actively deter people from putting in the effort to care, and be politically neutral enough to be only of interest to people who enjoy the fairytale for its own sake.

> 			`	Here at the Helsingor Family-Fun-Shopping-Center we are committed to celebrating our past! Which is why we renovated this old useless colonial structure and made it a vital part of our vibrant community!'` 

Author: I would say the best way to dissuade a narrative would be to denigrate it. _Only the loonies, REALLY, believe that we came from elves_. 

Also, this is also plaque in a very modern, very materialistic area, this is communicated to the pilot in multiple ways, and one of them is the plaque itself. Hence, why this is not the same as "deeply conservative Christian area", but the reverse. 
It could be 1000 words and more nuanced, but would certainly be a drag to read, and it also lessens the weight and purpose of the plaque. The value of the plaque is:

- It communicates to the player the "official story" regarding the God-Stones. 'Colonial Grafiti' goes hand-in-hand with the previous established lore on how the Deep handle the mystery of their origins.
- It communicates to the player what the prevalent attitude of this community is towards 'traditional' ideas. That they are, essentially, non-believers. 

An important theme in this mission is the tension between the old ways; the rural lower-class traditional people, and a more modern, more elite, more universal culture. A kind of tension like this exists everywhere, and certainly would exist in a nation that is actively trying to keep its populace from knowing the secret of its past, a 'secret' that traditional communities might still have access too.

I think you aren't considering the ramifications of what we learn in the "Deep: Questions mission" line. 
An archaeologist, with one piece of wood from one old building was able to find credible evidence that The Deep civilization pre-dates spaceflight by quite a bit. This means that any number of Deep researchers would have been able to easily discover the same. Which means its not a fairy-tale, its easily established fact for anyone who wants to do the science. Which means that in order for that information to be kept secret, The Deep has to be actively suppressing it. They have over 15 billion people on their worlds, and scientists would certainly be doing digs and dating things and curious about their origins, so just a passive" dissuading people from looking", is not nearly enough to cover that. Furthermore, as is demonstrated in the Deep: Questions mission, they actively restrict access to their region to outsiders, like the archaeologist, who could test the matter. Which means they either are in control of all their academics, or they are suppressing all their academics. Therefore every museum, every textbook, every research paper published, would out of necessity all support the lie. Otherwise, none of this would news to the archaeologist in that mission.
Which means every Deep child grows up learning the lie, and maybe their grandmothers tell them stories about elves and they like the elf-stories, they still believe that at its core its a bunch of nonsense.

> 			`	Here at the Helsingor Family-Fun-Shopping-Center we are committed to celebrating our past! Which is why we renovated this old useless colonial structure and made it a vital part of our vibrant community!'` 

Reviewer 4: You say `just a passive" dissuading people from looking", is not nearly enough to cover that.` but there are ways to intercept people who are looking, who are seeking grants for research etc. etc.

The problem as you so rightly point out is that the evidence actually is there to prove it really quite easily. Which means the most important feature in controlling the information, is _reducing the number of people who are willing to look._

Everything official needs to support the lie, I agree.
The problem is that, everything that supports the lie needs to do so in such a way as to not actively try and stoke the "loonies" into continuously trying to prove it wrong. If almost anyone can find proof, then the last thing you want to do is harass the "loonies" by actively denigrating them, because you'll have complete randoms coming out of the woodwork every two seconds with more proof. All just because you pointed your finger and said "no you can't". People are stubborn, if you attack them they'll fight back.

The more often people stick their head up and you shut them down, the greater the number of people who are going to pay attention, maybe look into it themselves. This is a problem if there's actual substance to it.

This is why in modern politics, the most popular way to escape a scandal is not to fight it, but to announce something else controversial and move on. You simply keep as much attention off of it as possible.

`Hence, why this is not the same as "deeply conservative Christian area", but the reverse.`
This is a wildly public location near a major spaceport, people who are not part of the modern elite would encounter it frequently enough for it to be an issue, and there's nothing quite like an outrage to motivate traditionalists with nothing better to do into organising a road trip for a large group all to stage a protest.

The best suppression is a message that says, "We don't believe, but we don't care if you do, because we have different things to care about." Because that encourages the people who do believe to just stay out of the way, and live their lives in peace, where they can't or won't encourage anyone to do their own research.

Active hostility to the message gives credence to it, the absolute last thing you want to do when suppressing a narrative is give credence to something that actually is credible.

> 			`	Here at the Helsingor Family-Fun-Shopping-Center we are committed to celebrating our past! Which is why we renovated this old useless colonial structure and made it a vital part of our vibrant community!'` 

Reviewer 4: `It could be 1000 words and more nuanced, but would certainly be a drag to read,`
Also, for the record, I'm not suggesting you actually write all that out, I'm suggesting you summarily state that that is the content of plaque and move on. Implicitly suggesting that the full contents are below the notice of the vast majority of people.

> 			`	Here at the Helsingor Family-Fun-Shopping-Center we are committed to celebrating our past! Which is why we renovated this old useless colonial structure and made it a vital part of our vibrant community!'` 

Author: I have edited the plaque description to make it less condescending.

					goto windblain
				`	"Asgard."`
					goto Asgard
			label windblain
			apply
				set "Nifel"
			`	"There is a nearly abandoned village called Hrithfjall next to a coastal God-Stone. Something about that place felt right. I believe that is where the Skadenga should migrate."`
			`	Hjlod nods. "Hrithfjall sounds like good place. I vill tell Ondurdis. Vhen you were gone thing, or assembly, vas held. Ondurdis called in all Skadenga from all villages across Nifel. You must find transportation for 2,200 folk. That is what Onderduis told me to tell you."`
			`	You frown and ask. "Did she provide money?"`
			`	Hjlod shakes her head. "No. Ve could not afford it. You must go to Valhalla, and convince head of Transport Union to support us."`
			`	Hjlod squeezes your shoulder. "Thank you for help. For finding us new home. Ve need it. I must report now to Ondurdis. Skade be vith you."`
			`	She turns and leaves.`
				accept
			label Asgard
			apply
				set "Asgard"
			`	"On Asgard there is a great market built around their God-Stone, and many new homes in need of people to fill them. The people there have lost their way and they need the Skadenga to remind them of who they were."`

> 			`	Hjlod nods. "The Skadenga have alvays been guides, pathfinders. I vill tell Ondurdis. Vhen you were gone thing, or assembly, vas held. Ondurdis called in all Skadenga from all villages across Nifel. You must find transportation for 2,200 folk. That is what Onderduis told me to tell you."`

Reviewer 4: Given the way this is presented in a somewhat positive light, and suddenly gives you an opinion that maybe they could have a calling here - the possibility of this outcome should really be foreshadowed earlier. Otherwise this option is entirely "doing it for the money" and suddenly "oh but here's a reason I've invented with no prior consideration to make you do it".

Author: I can change this. I just wanted to point out that this is said after you've made the decision. So the words here have no bearing on the decision making. The player should have already deduced for themselves what is the better option. This might give them hope that they can have their cake and eat it too, but they already know they've sold these people out.

Reviewer 4: `The player should have already deduced for themselves what is the better option.`
Oh absolutely, but it puts words into the players mouth, without any control of the player. This is generally frowned upon.
For narrative reasons I think it is still important, but the player should have some warning of exactly how they're intending to lie to these people.
Perhaps maybe when you first fine the box and the godstone the player could muse to themselves something to the effect of, "Maybe the Skadenga could find a purpose here?" or something. Just _something_ to foreshadow the nature of the lie before you actually get to it.

Author: I have altered the text on the plaque to make it less condescending.

Reviewer 4: Nice, I'm satisfied.

			
			
mission "Home for Skadenga 7"
	landing
	name "Evacuate <origin>"
	description `Escort a fleet of freighters, refitted for carrying primitives, to <destination>.`
	source "Nifel"
	destination "Asgard"
	to offer
		has "Home for Skadenga 6: done"
	on offer
		dialog
			`The fleet touches down just outside the over-crowded village of Udainsakr. The landscape is filled with the gathered Skadenga.`
			`	The Ondurdis approaches you. "Leaving is difficult. But all winters must come to end."`
			`	She watches as her people board the ships. "I must rejoin my folk. But thank you for this, and we shall speak again at our new home."`
	
	npc accompany save
		government "Deep Security"
		personality timid escort
		ship "Bactrian" "DTU Endeavor"
		ship "Bulk Freighter" "DTU Consistency"
		ship "Bulk Freighter" "DTU Dependable"
		ship "Freighter" "DTU Empathy"
	on visit
		dialog
			`You have reached <planet>, but the Transport Union ships have not all arrived yet. You should take off and wait for them to catch up with you.`
			
	on complete
		payment 850000
		event "skadenga death" 367
		conversation

> 			`For the occasion of the primitives' arrival Helsingor has remade itself into a mad circus. There are impromptu bands playing, random unplanned parades, teeming crowds of excited neighbors holding up signs welcoming the Skadenga into the 31st century. There is also every kind of emergency and civil personnel available here. Firemen, police, doctors, social workers; all claiming, testing, prodding and questioning the dazed immigrants.`

Reviewer 4: ```suggestion
			`For the occasion of the primitives' arrival, Helsingor has remade itself into a mad circus. There are impromptu bands playing, random unplanned parades, teeming crowds of excited neighbors holding up signs welcoming the Skadenga into the 31st century. There is also every kind of emergency and civil personnel available here. Firemen, police, doctors, social workers; all claiming, testing, prodding and questioning the dazed immigrants.`
```

			`	"Well done, Captain <last>. I've been authorized to tell you that my superiors have deposited 850,000 credits into your account, as was agreed upon earlier."`
			choice
				`	"Your superiors??"`
			`	Carla grins. "The Census and Culture Bureau. I am one of their field agents. They are most pleased with how you handled this. The Skadenga have been a problem for centuries, and you solved it."`
			`	She pats you on the back. "Anyways, I just wanted to pass that on. I have lots of work to do, thanks to you." She winks and wanders into the crowd.`
			`	You return to your ship to count your money.`
			
			
			
event "skadenga death"

mission "Skadenga Call 1"
	landing
	name "Check on Hjlod"
	description `Fly to <destination>, and see what Hjlod and the Skadenga need from you.`
	source
		government "Republic"
	destination "Asgard"
	to offer
		has "Home for Skadenga 7: done"
		has "event: skadenga death" 
	on offer
		dialog
			`	There is a new message in your inbox.`
			`	<first>, we need your help. Come to Asgard.`
			`			Signed, Hjlod`
	on complete
		dialog
			`Upon landing you receive another message in your inbox that simply says 'spaceport'.`

> mission "Skadenga Call 2"

Reviewer 4: This whole mission rubs me up wrong, The Deep is the standout example of possibly the only human society in ES that actually seems to care about the rights of people and the heritage of its past.

While I think it's pretty reasonable that the different people involved in trying to get them onto Asgard are leveraging a profit incentive to save the budget or just because it looks good for whatever politician is in charge, I don't think it's reasonable to go down the path of having then dismantled the community and abandoned the uncooperatives. If there's anyone left in the ES universe that appreciates the need to maintain communities to effectively assimilate them, it would be the Deep.

Perhaps this should be a hard-core idealist subset of them that just can't abide the "here to remind the people of their roots" narrative, and feel certain that their true place is elsewhere?

They could still be very disappointed in you, and maybe they're willing to burn down some modernist thing that particularly hurts their community? (Maybe the mall?) So you still have a source of conflict to report them on.

After all, a modern effort to "help" can still be misguided and dangerously ineffective without being actively malicious.

But perhaps just being able to take those hard-core idealists either back to Nifel, or on to Windblain might be viable, rather than rescuing them from a humanitarian nightmare.

Author: First, I am going to quote MZ here: 
_"It’s worth keeping in mind that pretty much every species or faction in Endless Sky lies to you, either because they have their own agenda or because they don’t know what the heck they’re talking about (or both). Sifting out what’s actually true is part of the fun, and my intention was to have different people tell you obviously contradictory stories about the same things._"

I would argue that my depiction of the Deep as flawed here falls directly in line with his world-building philosophy. If anything it brings them closer to MZ's ideals. To certain extent this also a small matter, you are dealing with a junior member of the Census and Culture Bureau, not its leadership, and in a case like this it is easy for a population of a few thousand to just fall between the cracks. The society, and maybe top leadership as a whole can certainly still be 'good guys'. But that doesn't mean that everyone throughout the entire bureaucracy is a saint. In the places where no one is looking, you're still going to have people being people. 

This is also is art imitating life, if you see how 'modern' societies deal with immigrants, or aboriginal populations (Native Americans/First Peoples in North America, Aboriginals in Australia, Sami in Scandinavia) etc. Even when these societies mean well, they often do significant damage, and someone in power nearly always has ulterior motives. 

The one thing though, is that we're dealing with 50 people out of 2200 who immigrated, I suppose we could show that many of those people did transition to their new culture fine. But, I am cautious of doing that for thematic reasons. The 'story' of this line is all about consequences. The player often makes decisions that effect people, and then flies away after the mission end without having face those decisions again. And this all about them meeting those consequences.

Reviewer 4: Yeah I'm not proposing that the rest of them all transitioned "fine", but just weren't badly enough off that the idealists would choose living in a dank alleyway wearing rags and evading law enforcement as a preferable alternative... like that's waaaaaaaayyyy darker than it needs to be. 

As an Australian, I'm quite familiar with the damage from meaning well but not understanding, that still happens today but in a more modern context. This present writing sounds like the 1970's "let's separate the children from their families and raise them civilised" approach, whereas the level I'm advocating for is the more modern version.
The modern version obviously still has many issues (public housing that separates them from their cultural practices so they choose to not live in it, education programs that teach in the dominant language to "help them learn" but leads to failure, etc.) but it's more 'callous neglect' or 'public relations posturing' and less 'actual humanitarian nightmare'.

Author: Alright, you've convinced me. I'll start thinking about a more nuanced and less grimdark approach to this =)

Author: I have re-written some of the mission. Hjlod now points out that most of the Skadenga are happy. The government's actions on Asgard are clearly from a help-first approach. I also re-wrote some of Carla's dialog to have her more sympathetic and less condescending. I think it now appears less malicious and more accidental/well-intentioned.

Reviewer 4: You actually went slightly nicer than I expected, but yes I think this is a much more appropriate tone. Threw in one suggestion in my new review but you can mark this thread as resolved.

	description `Escort your fleet carrying the Skadenga, to <destination>.`
	source "Nifel"
	destination "Windblain"
	to offer
		has "passenger space" > 2200
		has "refusedhroar"
	on offer
		dialog
			`Your fleet touches down just outside of the crowded village of Udainsakr. The landscape is filled with the gathered Skadenga.`
			`	The Ondurdis approaches you. "Leaving is difficult. But all winters must come to end."`
			`	She looks curiously at your fleet. "This I did not expect. These are not Transport Union, or Hroar's ships. These are yours? You brought your own fleet to help us?"`
			`	You smile and shrug, and let her know that sometimes you just got to do things yourself.`
			`	She shakes her head. "You are full of surprises, <first>. Perhaps, I'll surprise you someday as well."`
			`	Her eyes wander to her people who are busy embarking. "I must rejoin my folk. But thank you for this, and we shall speak again at our new home."`
			`	She leaves you with a warm smile in the wintry village.`
				accept
	
	on visit
		dialog
			`You have reached <planet>, but your ships have not all arrived yet. You should take off and wait for them to catch up with you.`	
		
	on complete

> 		event "Skadenga life" 367

Reviewer 3: ```suggestion
		event "skadenga life" 367
```

			`	"You are done looking? Ve are good to move to next stop?" Hjlod asks.`
			scene "scene/lonelyrock"
			choice
				`	"Yep, I don't need to look at this rock any longer."`
					goto byerock
				`	"Wait, is that part of a man's face carved into it? What is so special about this rock?"`
					goto manrock
			label manrock
			`	"That rock is supposed to celebrate man who discover Rigellian cure. He was called Hank, or Henry, or Bill, or something." Hjlod shrugs with evident disdain.`
			`	"The cure was found here, but back when tundra was melting and Nifel was a little varmer. They say this man discover it and they want to make grand statue for him. But it turn out Nifel not so great place for statue. Beside, no one like statue of liar."`
			`	Hjlod turns back to her controls and gets the Ice-Crawler moving forward again. "That is enough about stupid rock and stupider man."`
			`	"Now ve go to river."`
					goto riverlike
			label byerock
			`	"Now ve go to river."`
					goto riverlike
			label riverlike
			choice
				`	(The large oddly shaped rock falls out of your field of view as the Ice-Crawler turns away from it.)`
			scene "scene/snowfield"
					goto river
			label river

> 			`	The uneventful journey to the boiling ice river of Slidr is suddenly interrupted when the Ice-Crawler begins to shake wildly for a few moments, followed by a thunderous boom from somewhere outside.`
> 			`	After Hjlod regains control of the crawler, she immediately veers away from the noise. "That vas Nifel-quake, and loud noise was avalanche. This is not good."`
> 			`	Despite her calm demeanor she wields the Ice-Crawler with reckless abandon as she tries to avoid the rumbling snow. The machine protests with creaks and whirrs and pops as it goes faster than it has gone for hundreds of years.`
> 			`	"I think..." Hjlod starts to say when a mountain of snow crashes into the Ice-Crawler dragging it half a kilometer.`
> 			`	You and Hjlod hold tight to your seat-belts as you're tossed around. Eventually, the crawler comes to a rest on its' side.`

Reviewer 4: I would say it is _ok_ now, but could be improved further. Treat this as a range of suggestions that you can pick and choose from as it suits you. If you choose to reject this I won't shout about it. :P

```suggestion
			`	The uneventful journey to the boiling ice river of Slidr is suddenly interrupted when a deep tremor followed by a thunderous boom shakes the Ice-Crawler to its core and causes it to slip on the disturbed ice and snow.   Somewhere outside subsequent rumble starts to build rapidly.`
			`	After Hjlod regains control of the crawler, she immediately veers away from the noise. "That vas Nifel-quake, and loud noise was avalanche, or landslide, maybe both. This is not good."`
			`	Despite her calm demeanor she wields the Ice-Crawler with reckless abandon as she tries to avoid the rumbling snow. The machine protests with creaks and whirrs and pops as it strains to claw itself out of the way faster than it has probably gone for hundreds of years. The closer the rumbling gets, the more difficulty Hjlod seems to have retaining traction.`
			`	"I think..." Hjlod starts to say when a mountain of snow crashes into the Ice-Crawler with deafening force. All sense of travelling distance is lost as the snow blots out the windows and the crawler is dragged bumping and twisting over uneven terrain.`
			`	You and Hjlod hold tight to your seat-belts as you're tossed around until, eventually, the crawler grinds up against something reasonably solid and comes suddenly to a rest, partially on its' side.`
			`	A few moments later the rumbling finishes passing you by.`
```

			`	She ushers you past various barriers meant to keep a compliant public out. Finally, she leads you into a long hall-way filled with the detritus of life. Empty food containers, dirty blankets, other less savory items. People are also here, dirty, clumped together, wretched looking.`
			`	Hjlod turns back to you. "It safe to talk novhere. But ve must, so here must do."`
			choice
				`	"What do you want from me?"`
					goto what
				`	"Who are these people?"`
					goto who
			label what
			`	Hjlod looks at you severely. "From you? Everything. For that is what you took from us."`
			`	"I do not know vhy you did it. Vhy you brought us here. If Skade really let us down. Or if you ignored her." She growls.`
			`	"But it does not matter. All that matters is that ve leave, and that is vhy ve need you."`
					goto here
			label who
			`	Hjlod laughs and tears run down her cheeks. "Do you not recognize us? Our great savior? Ve are the Skadenga, and this is your handivwork."`

> 			`	She look away from you and continues. "The rest are not dead, no. But they have left us. Most are happy; happy to be fed, happy to be entertained, happy to forget vho they are, happy to no longer be Skadenga. But, those here cannot accept that. Ve refuse to become them."`

Reviewer 4: Slight tweak, leaning on perceived hypocrisy as a motivator.
```suggestion
			`	She look away from you and continues. "The rest are not dead, no. But they have left us. Most are happy; happy to be fed, happy to be entertained, happy to forget vho they are, happy be Skadenga only in name but not in practice. But, those here cannot accept that. Ve refuse to become them."`
```

			`	Hjlod grabs a raggedy duffle bag and a group gather around her. "The rest vill meet us at your ship. Ve move now."`
			choice
				`	(Follow Hjlod.)`
			`	Once you're back in the operational spaceport you and your group move quickly, while drawing occasional glances from curious on-lookers. A large group of disheveled Skaldenga are waiting for you when you reach your ship.`
			`	"Everyone on, quickly!" Hjlod barks at her people.`
			`	A few moments later they are all on board. "Asdis, headcount!" Hjlod says.`
			`	As you're waiting two more dirty and gaunt Skadenga show up. Asdis adds them to her count and then addresses Hjlod. "Ondurdis, 48 total."`
			`	Hjlod frowns. "It's Arnson's family isn't it? I told him he was too far."`
			`	A commotion boils up from a few terminals down, and you see some civil servants and local police officers heading your way.`
			`	Asdis is also watching. "They could still be coming."`
			`	"It's too late for them." Hjlod shuts the hatch. "<first>, if ever ve vere friends. Take off, now."`
			choice
				`	(Take off.)`
					goto takeoff
				`	(Power down your ship and open the hatch.)`

> 			`	You turn off your ship's ignition, pop open the hatch before Hjlod can react, and jump out. You find Carla Von Hausen, and a squad of police officers waiting for you.`

Reviewer 4: If they were already there by the time you open the hatch then there should've been some warning just before - just squeezes a few extra seconds into the sequence of events.
```suggestion
			`	You turn off your ship's ignition, pop open the hatch before Hjlod can react, and jump out. Carla Von Hausen, at least two civil service works, and a squad of police officers arrive within seconds.`
```

			`	'For heroic and selfless service to the ideals and prosperity of The Deep.'`
				decline

event "asgard hjlod"

mission "Skadenga Call 4"
	name "Hjlod's goodbye"
	invisible
	destination "Nifel"
	to offer
		has "Skadenga Call 2: done"
		has "event: nifelhjlod"
	on complete
		event "hjlod waits again" 360
		conversation
			`	A familiar woman is waiting for you outside your ship. Although it's been more than seven years since you've last seen her, you recognize her instantly. Her eyes are brighter than when last you parted, and she has the same lopsided grin that you remember.`
			`	"<first>, I saw your ship landing, and thought I vould say hello."`
			`	You ask her how she has been, and she tells you how the small community of Skadenga have rebuilt and thrived here. That there has even been a small amount of immigration to their tiny town. That word had gotten out about their desperate escape from, Asgard.`
			`	She tells you that she forgives you the past, and hopes you find whatever it is you are looking for among the stars. She even hugs you before she leaves.`

> 			`	As she walks away you wonder if you can ever forgive yourself.`

Reviewer 4: Suggested extra emphasis.
```suggestion
			`	As she walks away you wonder, though she might forgive you, if you can ever forgive yourself...`
```

			`	There is an expectant pause. You get the feeling they are waiting for you to either announce something or leave.`
			choice
				`	"I am looking for Hroar."`
			`	For a moment nothing happens and then a gruff voice spirals out from a dark corner. "Back here."`
			`	You walk to the back of the saloon and find a older man with long grey handlebar mustache seated with his back against the wall, and a bottle of whiskey on his otherwise empty table.`
			scene "scene/hroar"
			`	"Have a seat", He says.`
			`	Once seated, he places a gun on the table.`
			`	"So the Allen boys think they can send some green-horned killer to take me out?" Hroar barks a laugh. "Not likely going to happen."`
			`	With his free hand he takes a swig of his whiskey. "Here's what's going to happen. I am not going to kill you. Only maim you, a bit. And you tell the Allen boys that the next kid they send will come back even worse. We had a deal and I expect them to keep it."`
			choice
				`	"You got the wrong person. I don't even know who the Allen boys are.`
					goto next
				`	"I was not sent by the Allen whoevers, and if you even touch me, I will burn this entire planet to ash."`

> 					goto buffcheck
> 			label buffcheck

Reviewer 5: ```suggestion
```

Unless you have somewhere else that also sends the conversation to label buffcheck, this goto/label pair isn't needed. Whenever you have a choice, an option that doesn't specify a "goto" will always just go to the next line, just like the various times you've used it in the conversations previously.

It works both ways, too. For example, in the following block, choosing "you got the wrong person" will bypass the stuff listed under other choices to continue outside the choice with "we're all friends having a drink." While picking the gunfight option will send it to the combat label. (please excusing the indentation. GitHub text boxes don't handle tabs nicely)
```
		choice
			`	"You got the wrong person.`
			`	"Gunfight"`
				goto combat
			`We're all friends having a drink.`
				decline
			label combat
			`Time to have a shoot-out.`

```

			` 	She plunges into the snow and swim-climbs her way up it. You follow her, and though the cold is shocking, you are relieved when you quickly rise above it into the icy clear air above.`
			`	"Stay close," she says as she begins to trek through the snow. "Any path but mine is dangerous."`
			scene "scene/snowplain"
			choice
				`	(Walk in Hjlod's footsteps.)`
					goto travel
				`	(Meander around.)`
			`	Hjlod abruptly stops and glares at you. "You are fool. You vill certainly die, and you endanger me in process. This is no game, valk only vhere I valk or die.`
			choice
				`	(Walk in Hjlod's footsteps.)`
					goto travel
				`	(Continue to meander because you are sick of Hjlod telling you what to do.)`
			`	You walk in your looping meandering pattern, stepping where you want, and not paying any attention to the path Hjlod is taking.`
			`	You feel righteous vindication as the hours pass and nothing bad happens; proof that Hjlod was just being bossy. She should have no say over what you are doing; it's her fault you're stuck out here stranded on some trash-planet in the blinding cold in the first place.`

> 			`	At that thought you do some anger-stomping, really letting her know that you could care less about her worthless warnings. Which is, of course, when you hear a weird gurgling sizzle. You look down at your feet and see there is not packed snow beneath them, but that a grey-purple blob that has wrapped itself around your leg.`
> 			choice
> 				`	(Try and shake your leg free.)`
> 					goto free
> 				`	(Call out to Hjlod for help.)`
> 			`	Damn your pride. "Hjlod, help! Something's got my foot!" you cry.`
> 			`	Hjlod looks back at you with a dull expression on her face. "Is it purple-grey in color?"`

Reviewer: ```suggestion
			`	At that thought you do some anger-stomping, really letting her know that you could care less about her worthless warnings. Which is, of course, when you hear a weird gurgling sizzle. You look down at your feet and see there is not packed snow beneath them, but that a gray-purple blob that has wrapped itself around your leg.`
			choice
				`	(Try and shake your leg free.)`
					goto free
				`	(Call out to Hjlod for help.)`
			`	Damn your pride. "Hjlod, help! Something's got my foot!" you cry.`
			`	Hjlod looks back at you with a dull expression on her face. "Is it purple-gray in color?"`
```

			`	"Yes, yes! Quick, help. I don't know what to do."`
			`	"Vhat you do, is listen to Hjlod, and not make stompy circles in snow like spoilt child."`
			`	"Please, Hjlod!" You cry as the last of your boot dissolves away and you feel the slimy slug on your exposed foot.`
			`	"It is too late. Ice-Slug has you. Vhat you do now is die. Horribly." Hjlod turns and starts jogging away from you.`
			choice
				`	(Try and pull your leg free.)`
			label free
			`	Despite using all your strength, your leg won't budge. The sizzling noise continues and you are overwhelmed with a sharp horrific burning sensation. Your eyes water and you scream out in pain for help and mercy.`

> 			`	You keep pulling, trying to break free. The air thickens with the scent of your liquified foot, and something gives as you fall forward. You look down to see your leg missing from the knee down. You try to push it out of your mind as you desperately claw forward, trying to escape the grey-purple blob that is forming up behind you.`			
> 			`	Try as you might, you can't cry out Hjlod's name, instead your screams are savage, hollow noises. You reach for her, you reach for anyone, and it's too late. The Ice-Slug has caught up to you, you feel acidic tendrils wrapping around your remaining leg and lower torso. The pain is agonizing as your lower body begins to dissolve.`
> 			`	You look up one last desperate time, hoping to see Hjlod coming to rescue you. But instead you see a grey smudge on the horizon. A distant person who has long left you for dead. You are alone now with the Ice-Slug and the pain. You consider briefly your mistakes before you are dissolved and eaten.`

Reviewer: ```suggestion
			`	You keep pulling, trying to break free. The air thickens with the scent of your liquified foot, and something gives as you fall forward. You look down to see your leg missing from the knee down. You try to push it out of your mind as you desperately claw forward, trying to escape the gray-purple blob that is forming up behind you.`			
			`	Try as you might, you can't cry out Hjlod's name, instead your screams are savage, hollow noises. You reach for her, you reach for anyone, and it's too late. The Ice-Slug has caught up to you, you feel acidic tendrils wrapping around your remaining leg and lower torso. The pain is agonizing as your lower body begins to dissolve.`
			`	You look up one last desperate time, hoping to see Hjlod coming to rescue you. But instead you see a gray smudge on the horizon. A distant person who has long left you for dead. You are alone now with the Ice-Slug and the pain. You consider briefly your mistakes before you are dissolved and eaten.`
```

				die
			label travel
			`	The cold, monotonous trudging weighs on you as the hours pass. You're dimly aware of Nifel's two suns as they make their journey across the wide open sky.`
			`	Evening and exhaustion approach hand in hand, and as the last sun sets, you're not sure how much farther you can go. "There!" Hjlod breaks the silence of hours as she points at the weak flicker of light in the twilight landscape. "This is good," she admits hoarsely as you both stumble toward the light. "Ve vould have not survived night."`
			` 	The night air grows too cold. You slow as your body begins to shutdown and frostbite gnaws at your hands, feet and exposed arm. The light is still too distant, and you can barely move. You think you see the shadows of humanoid figures against the light. But then you stumble and fall. You crawl, for as long as you can, you crawl. And then nothing.`
			choice
				`	(Wake up.)`
			label villagearrival
			branch "bodyaches"
				has "put on jacket"
			`	The pale morning light of the first sun filters in through a synthetic window and onto your face. You try to escape back into sleep, but something feels off and groggily you peer out at your surroundings.`

> 			goto barracks

Reviewer 3: ```suggestion
				goto barracks
```

			label bodyaches
			`	The pale morning light of the first sun filters in through a synthetic window and onto your face. Your body aches and you try to sink back into sleep, but something feels off and you focus on your surroundings.`
			label barracks
			`	You're in a half-wooden, half-synthetic barracks, a strange mix of primitive earth-tech and old colonial architecture. That, in and of itself, isn't that alarming. What is, however, is the group of people gathered around your bed quietly staring and pointing at you.`
			choice
				`	"Excuse me?"`

> 			`	The crowd steps back, and an elderly woman steps forward. Her grey hair is rolled up into tight, elegant braids, and her face and hands are covered in faded, blue tattoos.`

Reviewer: ```suggestion
			`	The crowd steps back, and an elderly woman steps forward. Her gray hair is rolled up into tight, elegant braids, and her face and hands are covered in faded, blue tattoos.`
```

			`	The Ondurdis points a boney finger right at you. "First, you must discover which world will be our home. Then you must find us transportation. We will of course pay you in one way or another. Would you do this for us?"`
			choice
				`	"I'll do it. I won't let you guys down."`
					goto doit
				`	"No. This isn't my problem. I just need to get back to my ship."`
			`	The Ondurdis bows and steps back. "If that is your wish, then you must not be the one we wait for." With that, she and everyone else leave the barracks.`
			branch "useshuttle"
				has "put on jacket"
			`	You climb out of your bunk and head to the Ice-Crawler. From there, you head straight back to the spaceport. The trip takes a few hours and you spend some of it pondering the strange course of events - how you went from a life or death struggle against the elements to being some rustic community's savior.`
			`	"Oh, well." You say to yourself, "I can't save everyone."`

> 			goto onboard

Reviewer 3: ```suggestion
				goto onboard
```

			`	At this distance, you can see that the cairns have name-plates at their bases. Some are so faded and worn you cannot make out the names. Others are easier to read and appear to be family names. Myrlinga, Baning, and Amalung are the few you can read.`
			`	Beyond the cairns are the well-carved wooden pillars, or Setstokkr as the Ondurdis called them. They are each over 10 meters tall, and all covered entirely in runic script, images, and patterns. A few have unshorn branches that include their own carvings as well, while most are limbless. On one Setstokkr, you notice that a frequent carving within its entwined animals is similar in shape to the burn on your arm.`
			`	Finally, you reach the massive Goddess-Stone itself. You see now almost no part of the wide surface is bare. There are images within images, letters within letters. These exoctic letters bear no relationship to the runes found on the Setstokkr. These are truly alien.`
			choice
				`	(Head to the shuttle.)` 
			label shuttle
			branch "threefingers"
				has "put on jacket"
			`	A lone figure stands next to the idling shuttle. As you draw closer you recognize Hjlod.` 

> 			`	"I just vant to thank you." Hjlod's grey-blue eyes glitter in the morning light. "I alvays say Hjlod keep tourist safe. But yesterday you save Hjlod. I vill not forgot, know this."`

Reviewer: ```suggestion
			`	"I just vant to thank you." Hjlod's gray-blue eyes glitter in the morning light. "I alvays say Hjlod keep tourist safe. But yesterday you save Hjlod. I vill not forgot, know this."`
```

> 			goto lopsided

Reviewer 3: ```suggestion
				goto lopsided
```

		conversation
			`	Upon landing, you get recorded voice message from Hjlod. "<first>, ve're busy evacuating isolated Skadenga communities. Come back in few veeks vhen things settle down."`
			`	The message abruptly ends.`

event "nifel waiting"

mission "Home for Skadenga 2"
	name "Home for Skadenga"
	invisible
	landing
	to offer
		has "Home for Skadenga 1: active"
	source "Windblain"
	destination "Nifel"
	on offer
		conversation
			`In the high-atmosphere of <origin>, your ship's sensors pick up traces of the exotic Setstokkr matter. They pinpoint the material to the southern rim of a sparsely-populated island-continent, in the small coastal settlement of Hrithfjall.`
			`	Your ship lands just outside the settlement, and you leave with your handheld tracker keyed to the Setstokkr material. The trail goes cold once you enter the settlement. The hamlet Hrithfjall is little more than a few dozen wooden structures along a single overgrown road. The buildings themselves are well-crafted and finely designed, but many of them appear to be abandoned. There are no townsfolk about, but you notice recent footprints headed toward the shore, which you follow.` 
			scene "scene/seascape"

> 			`	The great seas of Windblain churn beneath the chalk-white cliffs and blue-green sky. As you scan the horizon, your eyes catch the glare of reflected light beyond a distant precipice. You head in that direction and find a figure seated at the precipice's edge. An aged man sits cross-legged, looking out into the sea and at a gleaming metal wall towering over the grey waves about a mile out from shore.`

Reviewer: ```suggestion
			`	The great seas of Windblain churn beneath the chalk-white cliffs and blue-green sky. As you scan the horizon, your eyes catch the glare of reflected light beyond a distant precipice. You head in that direction and find a figure seated at the precipice's edge. An aged man sits cross-legged, looking out into the sea and at a gleaming metal wall towering over the gray waves about a mile out from shore.`
```

			`	"Of course I knew." Jakob answers. "It is my business to know. The only people who live here anymore are employees of my firm and those hill-witches. Holding up both hands in a placating gesture, he adds, "And I really am sorry we could not extract you from that avalanche. But we did a cost-benefit analysis, and it just didn't make sense. You run a ship, a business, you understand. How often do you let crew die in boarding action to make a profit?"`
			`	Jakob puts his hands down. "Anyways, we hardly have the infrastructure to support rescue missions. Our financial team ran the numbers a few generations back, and we realized we could significantly increase our profit margins if we downsized our research division workforce to nothing, and instead focused on managing our existing patents.`
			`	"Since then we've been shutting things down, laying people off, and sending the rest offworld. Various services, including emergency ones, are only barely functional. It's much cheaper to shut this world down rather than try and keep managing it. That's why it will be completely uninhabited within 50 years."` 
			choice
				`	"You're abandoning an entire planet to make a profit?"`

> 			`	He leans forward with a sly look on his face. "Think about it. Instead of employing a worldwide workforce of 500,000 in the hope that we discover new pharmaceutical products, we instead employ 100 patent lawyers, and the rest of the profits go directly to the shareholders. We then raise the prices on our vast back catalogue of patented medicine, and everyone wins.`

Reviewer: ```suggestion
			`	He leans forward with a sly look on his face. "Think about it. Instead of employing a worldwide workforce of 500,000 in the hope that we discover new pharmaceutical products, we instead employ 100 patent lawyers, and the rest of the profits go directly to the shareholders. We then raise the prices on our vast back catalog of patented medicine, and everyone wins.`
```

			`	Either for profit, or out of a longing for home, many local communities promoted these ideas about the origins of these structures. Scientists have determined these structures were once large segments of the metal hulls of the original colony ships. To date, there has been some difficulty in identifying exactly what alloy they were constructed from.`
			`	Most worlds in The Deep have "Colonial Graffiti," and they are proud reminders of our enterprising colonial origins. They are sometimes known in older texts, and by early communities, as God-Stones and Elf-Stones.`
			``
			`	Here at the Helsingor Family-Fun-Shopping-Center, we are committed to celebrating our past! This is why we, in the spirit of graffiti being the cultural touchstone of the day, have renovated this colonial structure into a central communication piece and made it a vital part of our vibrant community!'`
			``

> 			`	You finish reading the plaque, and look at the dull grey metal wall and the inhuman things carved upon it. It looks nothing like a ship's hull.`

Reviewer: ```suggestion
			`	You finish reading the plaque, and look at the dull gray metal wall and the inhuman things carved upon it. It looks nothing like a ship's hull.`
```

			scene "scene/tower"
			`	Three youngish men adorned in trending clothes smile as you enter. "Ah, Captain <last>, what a pleasure to finally meet you. I am Kaiden, and these are my associates Braiden and Grayden. We are executives of the Transport Union. Obviously, we were made aware you would be coming and are looking forward toward creating a lasting and beneficial relationship with you and your... clients.`
			`	"We've heard that you have this inspiring idea to transport an entire backwater community from one world to another," Kaiden says.`
			`	"Really inspirational," Braiden chimes in.`
			`	"So inspiring," Grayden adds with an opaque hand gesture you think you've seen done by celebrities.`
			`	"At the Transport Union, we say that 'it's about the people in the transports,' which is why we're humbled and excited to offer you our support in this beautiful and inspiring humanitarian adventure of yours," Kaiden finishes. Braiden and Grayden both clap enthusiastically.`
			`	Kaiden waits for the ovation to subside then continues with a knowing smile. "So, tell us humble servants of the people: where to transport these poor souls?"`

> 			branch Asgard
> 					has "asgard"
> 			choice

Reviewer 3: ```suggestion
			branch asgard
					has "Asgard"
			choice
```

				`	"They need to go to Asgard."`
			`	Kaiden beams. "Brilliant. Just brilliant. I was just telling my colleagues here about what a great place Asgard would be for this tribe. The publicity will be huge. The universities will probably want to run various studies. There will be all sorts of government programs and initiatives there to help them accept the proper Deep culture and become productive citizens like the rest of us."`
			`	Grayden, Braiden and Kaiden all start shaking each others' hands and congratulating each other on being part of something so inspirational. After their celebration is over, Kaiden turns back to you. "Our transport fleet is at your disposal. They are being refitted at Mani in the Avior system."`
			`	The three stand up in unison. Grayden and Braiden leave, and Kaiden turns from the door. "We are so happy you stopped by today and gave us this opportunity to really give back to the needy communities out in our neighborhood." With that, they are all gone, and your escort returns to lead you out.`

mission "Home for Skadenga 5"
	name "Transport for Skadenga"
	landing
	description "Travel to <destination> to prepare the Transport Union Fleet."
	to offer

> 		has	"asgard"

Reviewer 3: ```suggestion
		has "Asgard"
```

			`	She reaches into her pocket and pulls out a small wooden box, similar to the one you saw displayed at Asgard years ago. "I have been thinking," Hjlod says. "Maybe all this vas vill of Skade. Maybe you took us to Asgard to make us stronger, to make us better. It is bitter lesson, but I understand it now."`
			` 	She holds out the box. "This is true of Skadenga. Only few ever found, and this I give to you. For saving my life, for bringing us back here, for making us stronger."`
			scene "outfit/skadetear"
			`	The box opens, displaying a small, blue glowing rock. "This is Tear of Skade. Alvays makes cold. Alvays. Useful on hot ship. Keep this vith you, and know Skadenga vith you too," Hjlod says, handing it to you.`	
			`	Her eyes water, and she wipes them. "Your story is forever chiseled on our stones."`
			`	She turns and leaves you there, holding a Tear of Skade, and shivering in Nifel's unforgiving cold.`

mission "The Book of Skadenga"
	invisible
	source
		government "Republic"
	to offer
		random < 10
		or
			has "Stones of Skadenga 4: done"
			and
				has "Skadenga Call 2: done"
				has "event: the book"
	on offer
		conversation

> 		`As you wander through the spaceport, something catches your eye at a kiosk selling gifts, trinkets, and other odds and ends.`
> 		`	There's an old-style book there, bound up in a ice-blue cover, titled: "The Skakdenga Saga: Against All Odds, by Hjlod Hrimnirsdottir."`
> 		choice
> 			`	(Ignore the book and move on.)`
> 				goto ignore
> 			`	(Pick the book up.)`

Reviewer 3: ```suggestion
			`As you wander through the spaceport, something catches your eye at a kiosk selling gifts, trinkets, and other odds and ends.`
			`	There's an old-style book there, bound up in a ice-blue cover, titled: "The Skakdenga Saga: Against All Odds, by Hjlod Hrimnirsdottir."`
			choice
				`	(Ignore the book and move on.)`
					goto ignore
				`	(Pick the book up.)`
```

			scene "scene/tower"
			`	Three youngish men adorned in trending clothes smile as you enter. "Ah, Captain <last>, what a pleasure to finally meet you. I am Kaiden, and these are my associates Braiden and Grayden. We are executives of the Transport Union. Obviously, we were made aware you would be coming and are looking forward toward creating a lasting and beneficial relationship with you and your... clients.`
			`	"We've heard that you have this inspiring idea to transport an entire backwater community from one world to another," Kaiden says.`
			`	"Really inspirational," Braiden chimes in.`
			`	"So inspiring," Grayden adds with an opaque hand gesture you think you've seen done by celebrities.`
			`	"At the Transport Union, we say that 'it's about the people in the transports,' which is why we're humbled and excited to offer you our support in this beautiful and inspiring humanitarian adventure of yours," Kaiden finishes. Braiden and Grayden both clap enthusiastically.`

> 			`	Kaiden waits for the ovation to subside then continues with a knowing smile. "So, tell us humble servants of the people: where to transport these poor doomed souls?"`

Reviewer: ```suggestion
			`	Kaiden waits for the ovation to subside then continues with a knowing smile. "So, tell us humble servants of the people: where to transport these poor souls?"`
```

			branch Asgard
					has "asgard"
			choice
				`	"That's fantastic news! The Skakenga will need transport to Windblain."`
				`	"They need to go to Windblain."`
			`	A hush falls over the room, and both Braiden and Grayden look to Kaiden with confused expressions on their faces.`

> 			`	"Wind. Blain." Kaiden says slowly. "Wiiiindblaaain. Are you sure that is the world these sad, simple, civilians wish to travel to? Maybe you got an invoice mixed up? Perhaps wrote the wrong name down?" Braiden and Grayden both silently nod.`

Reviewer: ```suggestion
			`	"Wind. Blain." Kaiden says slowly. "Wiiiindblaaain. Are you sure that is the world these simple people wish to travel to? Maybe you got an invoice mixed up? Perhaps wrote the wrong name down?" Braiden and Grayden both silently nod.`
```

			choice
				`	"That's fantastic news! The Skakenga will need transport to Asgard."`
				`	"They need to go to Asgard."`

> 			`	Kaiden beams. "Brilliant. Just brilliant. I was just telling my colleagues here about what a great place Asgard would be for a gaggle of prehistoric luddites. The publicity will be huge. The universities will probably want to run various studies. There will be all sorts of government programs and initiatives there to help them accept the proper Deep culture and become productive citizens like the rest of us."`

Reviewer: ```suggestion
			`	Kaiden beams. "Brilliant. Just brilliant. I was just telling my colleagues here about what a great place Asgard would be for this tribe. The publicity will be huge. The universities will probably want to run various studies. There will be all sorts of government programs and initiatives there to help them accept the proper Deep culture and become productive citizens like the rest of us."`
```

			`	"Well done, Captain <last>. I've been authorized to tell you that my superiors have deposited 850,000 credits into your account, as was agreed upon earlier."`
			choice
				`	"Your superiors??"`

> 			`	Carla grins. "The Census and Culture Bureau. I am one of their field agents. They are most pleased with how you handled this. Helping the Skadenga reintegrate into our culture is a big win for the Bureau, and the Skadenga."`

Reviewer 6: ```suggestion
			`	Carla grins. "The Census and Culture Bureau. I am one of their field agents. They are most pleased with how you handled this. Helping the Skadenga reintegrate into our culture is a big win for the Bureau and the Skadenga."`
```

mission "Stone-Bearers to Nifel"
	description "Take the mourning Skadenga to <destination> so that they can enact the rites of their dead."
	source "Windblain"
	stopover "Nifel"
	destination "Windblain"
	passengers 1 10 .9
	job
	repeat
	to offer
		has "Stones of Skadenga 3: offered"
		random < 30
	on accept
		dialog
			`The mourning Skadenga board your ship.`
	on visit
		dialog phrase "generic passenger on visit"
	on stopover
		dialog
			`The mourners take their small stones and head off to visit the cairns of their ancestors. After some time they return to your ship, happy to see the frozen-wastes of Nifel left behind.`
	on complete
		payment
		dialog

> 			`The Skadenga disembark and return to their lives on Windblain..`

Reviewer: ```suggestion
			`The Skadenga disembark and return to their lives on Windblain.`
```
